diff -Nurd cairo-1.12.2/config.h.in cairo-1.12.2/config.h.in
--- cairo-1.12.2/config.h.in	2012-04-29 21:50:29.000000000 +0300
+++ cairo-1.12.2/config.h.in	2012-08-05 00:40:58.000000000 +0300
@@ -261,6 +261,8 @@
 /* Define to 1 if you have zlib available */
 #undef HAVE_ZLIB
 
+#undef DBL_EPSILON
+
 /* Define to 1 if the system has the type `__uint128_t'. */
 #undef HAVE___UINT128_T
 
diff -Nurd cairo-1.12.2/configure cairo-1.12.2/configure
--- cairo-1.12.2/configure	2012-08-05 00:40:26.000000000 +0300
+++ cairo-1.12.2/configure	2012-08-05 00:40:06.000000000 +0300
@@ -20014,6 +20014,8 @@
   have_libz="no (requires zlib http://www.gzip.org/zlib/)"
 fi
 
+$as_echo "#define DBL_EPSILON (1e-7)" >> confdefs.h
+
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for dlsym in -ldl" >&5
 $as_echo_n "checking for dlsym in -ldl... " >&6; }
