diff -Nurd qt-everywhere-opensource-src-5.5.0/qt3d/qt3d.pro qt-everywhere-opensource-src-5.5.0/qt3d/qt3d.pro
--- qt-everywhere-opensource-src-5.5.0/qt3d/qt3d.pro	2015-06-29 23:11:42.000000000 +0300
+++ qt-everywhere-opensource-src-5.5.0/qt3d/qt3d.pro	2015-07-07 00:17:33.807747150 +0300
@@ -1,7 +1,8 @@
-#requires(qtHaveModule(opengl))
+requires(contains(QT_CONFIG, opengl))
 
 load(configure)
 qtCompileTest(assimp)
+
 load(qt_parts)
 
 #module_qt3d_tutorials.subdir = tutorials
