diff -Nurd ImageMagick-6.8.8-9/Makefile.am ImageMagick-6.8.8-9/Makefile.am
--- ImageMagick-6.8.8-9/Makefile.am	2014-04-04 01:22:00.513627320 +0300
+++ ImageMagick-6.8.8-9/Makefile.am	2014-04-04 01:21:45.233660595 +0300
@@ -40,9 +40,6 @@
 	$(UTILITIES_CLEANFILES) \
 	$(TESTS_CLEANFILES)
 
-bin_PROGRAMS = \
-	$(UTILITIES_PGMS)
-
 # Binary scripts
 bin_SCRIPTS = \
 	$(MAGICK_BIN_SCRPTS) \
diff -Nurd ImageMagick-6.8.8-9/Makefile.in ImageMagick-6.8.8-9/Makefile.in
--- ImageMagick-6.8.8-9/Makefile.in	2014-03-20 02:03:11.000000000 +0200
+++ ImageMagick-6.8.8-9/Makefile.in	2014-04-04 01:23:32.557426892 +0300
@@ -1793,12 +1793,7 @@
 	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(wand_libMagickWand_@MAGICK_MAJOR_VERSION@_@MAGICK_ABI_SUFFIX@_la_LDFLAGS) \
 	$(LDFLAGS) -o $@
-am__EXEEXT_1 = utilities/animate$(EXEEXT) utilities/compare$(EXEEXT) \
-	utilities/composite$(EXEEXT) utilities/conjure$(EXEEXT) \
-	utilities/convert$(EXEEXT) utilities/display$(EXEEXT) \
-	utilities/identify$(EXEEXT) utilities/import$(EXEEXT) \
-	utilities/mogrify$(EXEEXT) utilities/montage$(EXEEXT) \
-	utilities/stream$(EXEEXT)
+am__EXEEXT_1 =
 am__EXEEXT_2 = tests/validate$(EXEEXT) tests/drawtest$(EXEEXT) \
 	tests/wandtest$(EXEEXT)
 am__EXEEXT_3 = Magick++/demo/analyze$(EXEEXT) \
