diff -Nurd SDL_mixer-1.2.12/configure.in SDL_mixer-1.2.12/configure.in
--- SDL_mixer-1.2.12/configure.in	2013-06-16 01:23:55.555593465 +0300
+++ SDL_mixer-1.2.12/configure.in	2013-06-16 03:10:49.407452745 +0300
@@ -43,8 +43,7 @@
 AC_SUBST(LT_AGE)
 
 dnl Detect the canonical build and host environments
-AC_CONFIG_AUX_DIRS($srcdir/build-scripts)
-dnl AC_CANONICAL_HOST
+AC_CANONICAL_HOST
 
 dnl Check for tools
 AC_PROG_LIBTOOL
