diff -Nurd atk-1.24.0/atk/Makefile.am atk-1.24.0/atk/Makefile.am
--- atk-1.24.0/atk/Makefile.am	2008-09-22 12:34:50.000000000 +0300
+++ atk-1.24.0/atk/Makefile.am	2008-09-22 21:59:58.000000000 +0300
@@ -137,11 +137,11 @@
 # ---------- Win32 stuff ----------
 
 if OS_WIN32
-libatk_1_0_la_LDFLAGS += -export-symbols $(srcdir)/atk.def -no-undefined -Wl,atk-win32-res.o
+libatk_1_0_la_LDFLAGS += -export-symbols atk.def -no-undefined -Wl,atk-win32-res.o
 libatk_1_0_la_DEPENDENCIES = atk-win32-res.o atk.def
 
 install-def-file:
-	$(INSTALL) $(srcdir)/atk.def $(DESTDIR)$(libdir)/atk-1.0.def
+	$(INSTALL) atk.def $(DESTDIR)$(libdir)/atk-1.0.def
 uninstall-def-file:
 	-rm $(DESTDIR)$(libdir)/atk-1.0.def
 else
