diff -Nurd gtk+-3.4.4/gdk/fallback-c89.c gtk+-3.4.4/gdk/fallback-c89.c
--- gtk+-3.4.4/gdk/fallback-c89.c	2012-07-15 18:48:47.000000000 +0300
+++ gtk+-3.4.4/gdk/fallback-c89.c	2012-08-13 02:53:28.381643162 +0300
@@ -31,6 +31,7 @@
 }
 #endif
 
+#if 0
 #ifndef HAVE_ISINF
 /* Unfortunately MSVC does not have finite()
  * but it does have _finite() which is the same
@@ -42,3 +43,4 @@
   return (!_finite (x) && !_isnan (x));
 }
 #endif
+#endif
