diff -Nurd fontconfig-2.5.0/configure.in fontconfig-2.5.0/configure.in
--- fontconfig-2.5.0/configure.in	2007-11-14 04:54:45.000000000 +0200
+++ fontconfig-2.5.0/configure.in	2008-05-05 18:39:14.000000000 +0300
@@ -103,9 +103,11 @@
   AC_CHECK_PROGS(CC_FOR_BUILD, gcc cc)
 else
   CC_FOR_BUILD=$CC
+  CPPFLAGS_FOR_BUILD="$CPPFLAGS"
 fi
 AC_MSG_RESULT([$CC_FOR_BUILD])
 AC_SUBST(CC_FOR_BUILD)
+AC_SUBST(CPPFLAGS_FOR_BUILD)
 
 AC_MSG_CHECKING([for suffix of executable build tools])
 if test $cross_compiling = yes; then
diff -Nurd fontconfig-2.5.0/doc/Makefile.am fontconfig-2.5.0/doc/Makefile.am
--- fontconfig-2.5.0/doc/Makefile.am	2007-11-06 02:05:37.000000000 +0200
+++ fontconfig-2.5.0/doc/Makefile.am	2008-05-05 18:43:27.000000000 +0300
@@ -22,6 +22,7 @@
 # PERFORMANCE OF THIS SOFTWARE.
 
 CC = @CC_FOR_BUILD@
+CPPFLAGS = @CPPFLAGS_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 
diff -Nurd fontconfig-2.5.0/fc-arch/Makefile.am fontconfig-2.5.0/fc-arch/Makefile.am
--- fontconfig-2.5.0/fc-arch/Makefile.am	2007-10-25 07:39:14.000000000 +0300
+++ fontconfig-2.5.0/fc-arch/Makefile.am	2008-05-05 18:41:28.000000000 +0300
@@ -23,6 +23,7 @@
 #
 
 CC = @CC_FOR_BUILD@
+CPPFLAGS = @CPPFLAGS_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 
diff -Nurd fontconfig-2.5.0/fc-case/Makefile.am fontconfig-2.5.0/fc-case/Makefile.am
--- fontconfig-2.5.0/fc-case/Makefile.am	2007-10-25 07:42:36.000000000 +0300
+++ fontconfig-2.5.0/fc-case/Makefile.am	2008-05-05 18:49:03.000000000 +0300
@@ -23,6 +23,7 @@
 #
 
 CC = @CC_FOR_BUILD@
+CPPFLAGS = @CPPFLAGS_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 
diff -Nurd fontconfig-2.5.0/fc-glyphname/Makefile.am fontconfig-2.5.0/fc-glyphname/Makefile.am
--- fontconfig-2.5.0/fc-glyphname/Makefile.am	2007-10-25 07:40:17.000000000 +0300
+++ fontconfig-2.5.0/fc-glyphname/Makefile.am	2008-05-05 18:42:20.000000000 +0300
@@ -23,6 +23,7 @@
 #
 
 CC = @CC_FOR_BUILD@
+CPPFLAGS = @CPPFLAGS_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 
diff -Nurd fontconfig-2.5.0/fc-lang/Makefile.am fontconfig-2.5.0/fc-lang/Makefile.am
--- fontconfig-2.5.0/fc-lang/Makefile.am	2007-10-25 07:40:39.000000000 +0300
+++ fontconfig-2.5.0/fc-lang/Makefile.am	2008-05-05 18:44:01.000000000 +0300
@@ -23,6 +23,7 @@
 #
 
 CC = @CC_FOR_BUILD@
+CPPFLAGS = @CPPFLAGS_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 
