diff -Nurd gtk+-2.12.9/gdk/Makefile.am gtk+-2.12.9/gdk/Makefile.am
--- gtk+-2.12.9/gdk/Makefile.am	2008-05-11 00:02:47.000000000 +0300
+++ gtk+-2.12.9/gdk/Makefile.am	2008-05-11 00:06:10.000000000 +0300
@@ -161,11 +161,11 @@
 libgdk_win32_2_0_la_LIBADD = win32/libgdk-win32.la $(GDK_DEP_LIBS) \
   $(top_builddir)/gdk-pixbuf/libgdk_pixbuf-$(GTK_API_VERSION).la
 libgdk_win32_2_0_la_DEPENDENCIES = win32/libgdk-win32.la win32/rc/gdk-win32-res.o gdk.def
-libgdk_win32_2_0_la_LDFLAGS = -Wl,win32/rc/gdk-win32-res.o -export-symbols gdk.def $(LDADD)
+libgdk_win32_2_0_la_LDFLAGS = -Wl,win32/rc/gdk-win32-res.o -export-symbols $(srcdir)/gdk.def $(LDADD)
 
 if OS_WIN32
 install-def-file: gdk.def
-	$(INSTALL) gdk.def $(DESTDIR)$(libdir)/gdk-win32-2.0.def
+	$(INSTALL) $(srcdir)/gdk.def $(DESTDIR)$(libdir)/gdk-win32-2.0.def
 uninstall-def-file:
 	-rm $(DESTDIR)$(libdir)/gdk-win32-2.0.def
 else
@@ -189,8 +189,10 @@
 uninstall-ms-lib:
 endif
 
+# This places generated .def to source directory, since it is expected there
+# (one from distributed tarball is)
 gdk.def: gdk.symbols
-	(echo -e EXPORTS; $(CPP) -P -DALL_FILES -DGDK_WINDOWING_WIN32 -DINCLUDE_VARIABLES - <$(srcdir)/gdk.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > gdk.def
+	(echo -e EXPORTS; $(CPP) -P -DALL_FILES -DGDK_WINDOWING_WIN32 -DINCLUDE_VARIABLES - <$(srcdir)/gdk.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > $(srcdir)/gdk.def
 
 gdkalias.h: gdk.symbols
 	 $(PERL) $(srcdir)/makegdkalias.pl < $(srcdir)/gdk.symbols > gdkalias.h
diff -Nurd gtk+-2.12.9/gdk-pixbuf/Makefile.am gtk+-2.12.9/gdk-pixbuf/Makefile.am
--- gtk+-2.12.9/gdk-pixbuf/Makefile.am	2008-05-11 00:02:47.000000000 +0300
+++ gtk+-2.12.9/gdk-pixbuf/Makefile.am	2008-05-11 00:05:44.000000000 +0300
@@ -7,7 +7,7 @@
 
 if OS_WIN32
 gdk_pixbuf_def = gdk_pixbuf.def
-gdk_pixbuf_symbols = -export-symbols gdk_pixbuf.def
+gdk_pixbuf_symbols = -export-symbols $(srcdir)/gdk_pixbuf.def
 
 gdk_pixbuf_win32_res = gdk_pixbuf-win32-res.o
 gdk_pixbuf_win32_res_ldflag = -Wl,gdk_pixbuf-win32-res.o
@@ -16,7 +16,7 @@
 	$(WINDRES) gdk_pixbuf.rc $@
 
 install-def-file:
-	$(INSTALL) gdk_pixbuf.def $(DESTDIR)$(libdir)/gdk_pixbuf-$(GTK_API_VERSION).def
+	$(INSTALL) $(srcdir)/gdk_pixbuf.def $(DESTDIR)$(libdir)/gdk_pixbuf-$(GTK_API_VERSION).def
 uninstall-def-file:
 	-rm $(DESTDIR)$(libdir)/gdk_pixbuf-$(GTK_API_VERSION).def
 else
@@ -40,8 +40,10 @@
 uninstall-ms-lib:
 endif
 
+# This places generated .def to source directory, since it is expected there
+# (one from distributed tarball is)
 gdk_pixbuf.def: gdk-pixbuf.symbols
-	(echo -e EXPORTS; $(CPP) -P -DINCLUDE_VARIABLES -DG_OS_WIN32 -DALL_FILES - <$(srcdir)/gdk-pixbuf.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > gdk_pixbuf.def
+	(echo -e EXPORTS; $(CPP) -P -DINCLUDE_VARIABLES -DG_OS_WIN32 -DALL_FILES - <$(srcdir)/gdk-pixbuf.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > $(srcdir)/gdk_pixbuf.def
 
 gdk-pixbuf-alias.h: gdk-pixbuf.symbols
 	 $(PERL) $(srcdir)/makegdkpixbufalias.pl < $(srcdir)/gdk-pixbuf.symbols > gdk-pixbuf-alias.h
diff -Nurd gtk+-2.12.9/gtk/Makefile.am gtk+-2.12.9/gtk/Makefile.am
--- gtk+-2.12.9/gtk/Makefile.am	2008-05-11 00:02:47.000000000 +0300
+++ gtk+-2.12.9/gtk/Makefile.am	2008-05-11 00:06:53.000000000 +0300
@@ -51,7 +51,7 @@
 
 if OS_WIN32
 gtk_def = gtk.def
-gtk_win32_symbols = -export-symbols gtk.def
+gtk_win32_symbols = -export-symbols $(srcdir)/gtk.def
 
 gtk_win32_res = gtk-win32-res.o
 gtk_win32_res_ldflag = -Wl,gtk-win32-res.o
@@ -60,7 +60,7 @@
 	$(WINDRES) gtk-win32.rc $@
 
 install-def-file: gtk.def
-	$(INSTALL) gtk.def $(DESTDIR)$(libdir)/gtk-win32-2.0.def
+	$(INSTALL) $(srcdir)/gtk.def $(DESTDIR)$(libdir)/gtk-win32-2.0.def
 uninstall-def-file:
 	-rm $(DESTDIR)$(libdir)/gtk-win32-2.0.def
 else
@@ -84,8 +84,10 @@
 uninstall-ms-lib:
 endif
 
+# This places generated .def to source directory, since it is expected there
+# (one from distributed tarball is)
 gtk.def: gtk.symbols
-	(echo -e EXPORTS; $(CPP) -P -DINCLUDE_VARIABLES -DG_OS_WIN32 -DALL_FILES - <$(srcdir)/gtk.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > gtk.def
+	(echo -e EXPORTS; $(CPP) -P -DINCLUDE_VARIABLES -DG_OS_WIN32 -DALL_FILES - <$(srcdir)/gtk.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > $(srcdir)/gtk.def
 
 gtkalias.h: gtk.symbols
 	  $(PERL) $(srcdir)/makegtkalias.pl < $(srcdir)/gtk.symbols > gtkalias.h
