diff -Nurd binutils-2.22/configure binutils-2.22/configure
--- binutils-2.22/configure	2012-04-18 01:44:42.000000000 +0300
+++ binutils-2.22/configure	2012-04-18 01:44:56.000000000 +0300
@@ -3664,7 +3664,7 @@
 if test "${build}" != "${host}" ; then
   AR_FOR_BUILD=${AR_FOR_BUILD-ar}
   AS_FOR_BUILD=${AS_FOR_BUILD-as}
-  CC_FOR_BUILD=${CC_FOR_BUILD-gcc}
+  CC_FOR_BUILD="LIBRARY_PATH=${CROSSER_NAT_LIBP} ${CC_FOR_BUILD-gcc}"
   CXX_FOR_BUILD=${CXX_FOR_BUILD-g++}
   GCJ_FOR_BUILD=${GCJ_FOR_BUILD-gcj}
   GFORTRAN_FOR_BUILD=${GFORTRAN_FOR_BUILD-gfortran}
diff -Nurd binutils-2.22/configure.ac binutils-2.22/configure.ac
--- binutils-2.22/configure.ac	2012-04-18 01:44:42.000000000 +0300
+++ binutils-2.22/configure.ac	2012-04-18 01:44:28.000000000 +0300
@@ -1081,7 +1081,7 @@
 if test "${build}" != "${host}" ; then
   AR_FOR_BUILD=${AR_FOR_BUILD-ar}
   AS_FOR_BUILD=${AS_FOR_BUILD-as}
-  CC_FOR_BUILD=${CC_FOR_BUILD-gcc}
+  CC_FOR_BUILD="LIBRARY_PATH=${CROSSER_NAT_LIBP} ${CC_FOR_BUILD-gcc}"
   CXX_FOR_BUILD=${CXX_FOR_BUILD-g++}
   GCJ_FOR_BUILD=${GCJ_FOR_BUILD-gcj}
   GFORTRAN_FOR_BUILD=${GFORTRAN_FOR_BUILD-gfortran}
