diff -Nurd fontconfig-2.17.1/Makefile.in fontconfig-2.17.1/Makefile.in
--- fontconfig-2.17.1/Makefile.in	2025-07-02 07:53:19.322247000 +0300
+++ fontconfig-2.17.1/Makefile.in	2026-08-16 12:11:14.309550099 +0300
@@ -221,7 +221,7 @@
   done | $(am__uniquify_input)`
 DIST_SUBDIRS = fontconfig fc-case fc-lang src fc-cache fc-cat \
 	fc-conflist fc-list fc-match fc-pattern fc-query fc-scan \
-	fc-validate conf.d its po po-conf test doc
+	fc-validate conf.d its po po-conf doc
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/config.h.in \
 	$(srcdir)/fontconfig-zip.in $(srcdir)/fontconfig.pc.in \
 	ABOUT-NLS AUTHORS COPYING ChangeLog INSTALL NEWS README.md \
@@ -475,7 +475,7 @@
 top_srcdir = @top_srcdir@
 SUBDIRS = fontconfig fc-case fc-lang src fc-cache fc-cat fc-conflist \
 	fc-list fc-match fc-pattern fc-query fc-scan fc-validate \
-	conf.d its po po-conf test $(am__append_1)
+	conf.d its po po-conf $(am__append_1)
 ACLOCAL_AMFLAGS = -I m4
 MESON_FILES = \
 	conf.d/link_confs.py \
diff -Nurd fontconfig-2.17.1/configure fontconfig-2.17.1/configure
--- fontconfig-2.17.1/configure	2025-07-02 07:53:18.850246400 +0300
+++ fontconfig-2.17.1/configure	2026-08-16 12:11:57.622407525 +0300
@@ -22068,7 +22068,7 @@
 
 
 
-ac_config_files="$ac_config_files Makefile fontconfig/Makefile fc-lang/Makefile fc-case/Makefile src/Makefile conf.d/Makefile fc-cache/Makefile fc-cat/Makefile fc-conflist/Makefile fc-list/Makefile fc-match/Makefile fc-pattern/Makefile fc-query/Makefile fc-scan/Makefile fc-validate/Makefile fontconfig/fontconfig.h doc/Makefile doc/cache-version.sgml doc/version.sgml its/Makefile po/Makefile.in po-conf/Makefile.in test/Makefile fontconfig.pc fontconfig-zip"
+ac_config_files="$ac_config_files Makefile fontconfig/Makefile fc-lang/Makefile fc-case/Makefile src/Makefile conf.d/Makefile fc-cache/Makefile fc-cat/Makefile fc-conflist/Makefile fc-list/Makefile fc-match/Makefile fc-pattern/Makefile fc-query/Makefile fc-scan/Makefile fc-validate/Makefile fontconfig/fontconfig.h doc/Makefile doc/cache-version.sgml doc/version.sgml its/Makefile po/Makefile.in po-conf/Makefile.in fontconfig.pc fontconfig-zip"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
