diff -Nurd fontconfig-2.8.0/configure.in fontconfig-2.8.0/configure.in
--- fontconfig-2.8.0/configure.in	2012-04-17 00:39:01.000000000 +0300
+++ fontconfig-2.8.0/configure.in	2012-04-17 01:09:50.000000000 +0300
@@ -104,6 +104,7 @@
 AC_MSG_CHECKING([for a C compiler for build tools])
 if test $cross_compiling = yes; then
   AC_CHECK_PROGS(CC_FOR_BUILD, gcc cc)
+  CC_FOR_BUILD="LIBRARY_PATH=$CROSSER_NAT_LIBP $CC_FOR_BUILD"
 else
   CC_FOR_BUILD=$CC
 fi
