diff -Nurd gdk-pixbuf-2.36.12/configure gdk-pixbuf-2.36.12/configure
--- gdk-pixbuf-2.36.12/configure	2018-04-07 21:42:22.000000000 +0300
+++ gdk-pixbuf-2.36.12/configure	2018-08-13 18:33:44.192733270 +0300
@@ -20776,7 +20776,7 @@
 # Output commands
 ##################################################
 
-ac_config_files="$ac_config_files Makefile gdk-pixbuf-2.0.pc gdk-pixbuf/Makefile gdk-pixbuf/gdk_pixbuf.rc gdk-pixbuf/gdk-pixbuf-features.h gdk-pixbuf/pixops/Makefile win32/Makefile win32/vs9/Makefile win32/vs9/gdk-pixbuf-version-paths.vsprops win32/vs10/Makefile win32/vs10/gdk-pixbuf-version-paths.props win32/vs11/Makefile win32/vs12/Makefile win32/vs14/Makefile win32/vs15/Makefile docs/Makefile docs/reference/Makefile docs/reference/gdk-pixbuf/Makefile docs/reference/gdk-pixbuf/version.xml po/Makefile.in tests/Makefile thumbnailer/Makefile contrib/Makefile contrib/gdk-pixbuf-xlib/Makefile contrib/gdk-pixbuf-xlib/gdk-pixbuf-xlib-2.0.pc config.h.win32"
+ac_config_files="$ac_config_files Makefile gdk-pixbuf-2.0.pc gdk-pixbuf/Makefile gdk-pixbuf/gdk_pixbuf.rc gdk-pixbuf/gdk-pixbuf-features.h gdk-pixbuf/pixops/Makefile win32/Makefile win32/vs9/Makefile win32/vs9/gdk-pixbuf-version-paths.vsprops win32/vs10/Makefile win32/vs10/gdk-pixbuf-version-paths.props win32/vs11/Makefile win32/vs12/Makefile win32/vs14/Makefile win32/vs15/Makefile docs/Makefile docs/reference/Makefile docs/reference/gdk-pixbuf/Makefile docs/reference/gdk-pixbuf/version.xml po/Makefile.in tests/Makefile contrib/Makefile contrib/gdk-pixbuf-xlib/Makefile contrib/gdk-pixbuf-xlib/gdk-pixbuf-xlib-2.0.pc config.h.win32"
 
 
 cat >confcache <<\_ACEOF
diff -Nurd gdk-pixbuf-2.36.12/Makefile.in gdk-pixbuf-2.36.12/Makefile.in
--- gdk-pixbuf-2.36.12/Makefile.in	2018-04-07 21:42:24.000000000 +0300
+++ gdk-pixbuf-2.36.12/Makefile.in	2018-08-13 18:34:02.496626833 +0300
@@ -450,7 +450,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = gdk-pixbuf po docs thumbnailer tests contrib win32
+SUBDIRS = gdk-pixbuf po docs tests contrib win32
 meson_files = \
 	meson.build \
 	meson_options.txt \
