diff -Nurd glew-2.0.0/config/Makefile.linux-mingw-w64 glew-2.0.0/config/Makefile.linux-mingw-w64
--- glew-2.0.0/config/Makefile.linux-mingw-w64	2017-01-14 23:07:18.165614800 +0200
+++ glew-2.0.0/config/Makefile.linux-mingw-w64	2017-01-14 23:07:50.385856291 +0200
@@ -5,13 +5,12 @@
 #
 
 NAME  := glew32
-CC    := i686-w64-mingw32-gcc
-LD    := i686-w64-mingw32-ld
+CC    := $(CROSSER_TARGET)-gcc
+LD    := $(CROSSER_TARGET)-ld
 LN    :=
 STRIP :=
 LDFLAGS.GL = -lopengl32 -lgdi32 -luser32 -lkernel32
 CFLAGS.EXTRA  += -fno-builtin -fno-stack-protector
-LDFLAGS.EXTRA += -nostdlib
 WARN = -Wall -W
 POPT = -O2
 BIN.SUFFIX = .exe
diff -Nurd glew-2.0.0/Makefile glew-2.0.0/Makefile
--- glew-2.0.0/Makefile	2017-01-14 23:07:18.169614830 +0200
+++ glew-2.0.0/Makefile	2017-01-14 23:07:01.069486742 +0200
@@ -31,7 +31,7 @@
 include config/version
 
 SHELL = /bin/sh
-SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/[0-9\.]//g;')
+SYSTEM = linux-mingw-w64
 SYSTEM.SUPPORTED = $(shell test -f config/Makefile.$(SYSTEM) && echo 1)
 
 ifeq ($(SYSTEM.SUPPORTED), 1)
@@ -40,8 +40,8 @@
 $(error "Platform '$(SYSTEM)' not supported")
 endif
 
-GLEW_PREFIX ?= /usr
-GLEW_DEST ?= /usr
+GLEW_PREFIX ?= $(DLLSPREFIX)/usr
+GLEW_DEST ?= $(DLLSPREFIX)/usr
 BINDIR    ?= $(GLEW_DEST)/bin
 LIBDIR    ?= $(GLEW_DEST)/lib
 INCDIR    ?= $(GLEW_DEST)/include/GL
@@ -66,10 +66,10 @@
 # To disable symlinks:
 #   - use LN= on gmake command-line
 
-AR       ?= ar
+AR       ?= $(CROSSER_TARGET)-ar
 ARFLAGS  ?= cr
 INSTALL  ?= install
-STRIP    ?= strip
+STRIP    ?= $(CROSSER_TARGET)-strip
 RM       ?= rm -f
 LN       ?= ln -sf
 UNIX2DOS ?= unix2dos -q
