diff -Nurd glibc-ports-2.7/sysdeps/unix/sysv/linux/arm/ioperm.c glibc-ports-2.7/sysdeps/unix/sysv/linux/arm/ioperm.c
--- glibc-ports-2.7/sysdeps/unix/sysv/linux/arm/ioperm.c	2005-06-10 14:12:09.000000000 +0300
+++ glibc-ports-2.7/sysdeps/unix/sysv/linux/arm/ioperm.c	2008-08-19 02:18:17.000000000 +0300
@@ -45,7 +45,6 @@
 #include <sys/mman.h>
 
 #include <linux/version.h>
-#include <asm/page.h>
 #include <sys/sysctl.h>
 
 #define PATH_ARM_SYSTYPE	"/etc/arm_systype"
