diff -Nurd glibc-2.11.2/nptl/sysdeps/pthread/pt-initfini.c glibc-2.11.2/nptl/sysdeps/pthread/pt-initfini.c
--- glibc-2.11.2/nptl/sysdeps/pthread/pt-initfini.c	2010-05-19 23:38:20.000000000 +0300
+++ glibc-2.11.2/nptl/sysdeps/pthread/pt-initfini.c	2010-06-22 11:48:25.000000000 +0300
@@ -45,6 +45,11 @@
 /* Embed an #include to pull in the alignment and .end directives. */
 asm ("\n#include \"defs.h\"");
 
+asm ("\n#if defined __i686 && defined __ASSEMBLER__");
+asm ("\n#undef __i686");
+asm ("\n#define __i686 __i686");
+asm ("\n#endif");
+
 /* The initial common code ends here. */
 asm ("\n/*@HEADER_ENDS*/");
 
diff -Nurd glibc-2.11.2/sysdeps/unix/sysv/linux/i386/sysdep.h glibc-2.11.2/sysdeps/unix/sysv/linux/i386/sysdep.h
--- glibc-2.11.2/sysdeps/unix/sysv/linux/i386/sysdep.h	2010-05-19 23:38:20.000000000 +0300
+++ glibc-2.11.2/sysdeps/unix/sysv/linux/i386/sysdep.h	2010-06-22 11:47:41.000000000 +0300
@@ -29,6 +29,10 @@
 #include <dl-sysdep.h>
 #include <tls.h>
 
+#if defined __i686 && defined __ASSEMBLER__
+#undef __i686
+#define __i686 __i686
+#endif
 
 /* For Linux we can use the system call table in the header file
 	/usr/include/asm/unistd.h
