diff -Nurd gnome-icon-theme-3.12.0/Makefile.in gnome-icon-theme-3.12.0/Makefile.in
--- gnome-icon-theme-3.12.0/Makefile.in	2014-03-25 06:37:01.000000000 +0200
+++ gnome-icon-theme-3.12.0/Makefile.in	2018-03-13 15:02:23.461027821 +0200
@@ -947,9 +947,7 @@
 	fi
 
 install-data-hook:
-	if test -z "$(DESTDIR)" ; then \
-		$(GTK_UPDATE_ICON_CACHE) -q $(DESTDIR)$(themedir); \
-	fi
+
 
 uninstall-hook:
 	@rm -f  $(DESTDIR)$(themedir)/icon-theme.cache
