diff -Nurd libgpg-error-1.10/src/Makefile.am libgpg-error-1.10/src/Makefile.am
--- libgpg-error-1.10/src/Makefile.am	2012-04-10 23:58:55.000000000 +0300
+++ libgpg-error-1.10/src/Makefile.am	2012-04-10 23:59:34.000000000 +0300
@@ -168,7 +168,7 @@
 # It is correct to use $(CC_FOR_BUILD) here.  We want to run the
 # program at build time.
 mkerrcodes: mkerrcodes.c mkerrcodes.h Makefile
-	$(CC_FOR_BUILD) -I. -I$(srcdir) -o $@ $(srcdir)/mkerrcodes.c
+	LIBRARY_PATH="$(CROSSER_NAT_LIBP)" $(CC_FOR_BUILD) -I. -I$(srcdir) -o $@ $(srcdir)/mkerrcodes.c
 
 if HAVE_W32CE_SYSTEM
 # It is correct to use $(CC_FOR_BUILD) here.  We want to run the
