diff -Nurd graphene-1.4.0/src/graphene-private.h graphene-1.4.0/src/graphene-private.h
--- graphene-1.4.0/src/graphene-private.h	2016-05-17 12:36:51.000000000 +0300
+++ graphene-1.4.0/src/graphene-private.h	2016-10-21 06:06:37.302078845 +0300
@@ -31,6 +31,10 @@
 #include <math.h>
 #include <float.h>
 
+#ifndef FLT_EPSILON
+#define FLT_EPSILON __FLT_EPSILON__
+#endif
+
 #define GRAPHENE_FLOAT_EPSILON  FLT_EPSILON
 
 #ifndef MIN
diff -Nurd graphene-1.4.0/src/graphene-simd4x4f.h graphene-1.4.0/src/graphene-simd4x4f.h
--- graphene-1.4.0/src/graphene-simd4x4f.h	2016-10-21 06:02:10.498498975 +0300
+++ graphene-1.4.0/src/graphene-simd4x4f.h	2016-10-21 06:05:02.330225654 +0300
@@ -29,6 +29,10 @@
 #include <math.h>
 #include <float.h>
 
+#ifndef FLT_EPSILON
+#define FLT_EPSILON __FLT_EPSILON__
+#endif
+
 GRAPHENE_BEGIN_DECLS
 
 /**
