diff -Nurd graphene-1.6.2/configure graphene-1.6.2/configure
--- graphene-1.6.2/configure	2018-08-13 22:11:40.991881364 +0300
+++ graphene-1.6.2/configure	2018-08-13 22:11:54.151805497 +0300
@@ -16972,7 +16972,7 @@
 fi
 
 
-for ac_func in aligned_alloc posix_memalign memalign
+for ac_func in aligned_alloc posix_memalign memalign _aligned_malloc
 do :
   as_ac_var=`$as_echo "ac_cv_func_$ac_func" | $as_tr_sh`
 ac_fn_c_check_func "$LINENO" "$ac_func" "$as_ac_var"
diff -Nurd graphene-1.6.2/src/config.h.in graphene-1.6.2/src/config.h.in
--- graphene-1.6.2/src/config.h.in	2018-02-23 01:00:46.000000000 +0200
+++ graphene-1.6.2/src/config.h.in	2018-08-13 22:12:30.827594404 +0300
@@ -3,6 +3,9 @@
 /* Define to 1 if you have the `aligned_alloc' function. */
 #undef HAVE_ALIGNED_ALLOC
 
+/* Define to 1 if you have the `_aligned_malloc' function. */
+#undef HAVE__ALIGNED_MALLOC
+
 /* Define to 1 if you have the <dlfcn.h> header file. */
 #undef HAVE_DLFCN_H
 
