diff -Nurd gtk-3.96.0/gtk/gtkimcontextime.c gtk-3.96.0/gtk/gtkimcontextime.c
--- gtk-3.96.0/gtk/gtkimcontextime.c	2019-11-22 08:38:48.246787928 +0200
+++ gtk-3.96.0/gtk/gtkimcontextime.c	2019-11-22 08:39:12.366919062 +0200
@@ -881,7 +881,7 @@
 
   widget = gtk_root_get_for_surface (context_ime->client_surface);
   if (!widget)
-    return
+    return;
 
   hwnd = gdk_win32_surface_get_impl_hwnd (context_ime->client_surface);
   himc = ImmGetContext (hwnd);
