diff -Nurd gtk-3.96.0/gdk/win32/gdkcairocontext-win32.c gtk-3.96.0/gdk/win32/gdkcairocontext-win32.c
--- gtk-3.96.0/gdk/win32/gdkcairocontext-win32.c	2019-05-07 06:27:29.000000000 +0300
+++ gtk-3.96.0/gdk/win32/gdkcairocontext-win32.c	2019-07-19 16:09:45.597730074 +0300
@@ -27,7 +27,7 @@
 
 #include <cairo-win32.h>
 
-#include <Windows.h>
+#include <windows.h>
 
 G_DEFINE_TYPE (GdkWin32CairoContext, gdk_win32_cairo_context, GDK_TYPE_CAIRO_CONTEXT)
 
diff -Nurd gtk-3.96.0/gdk/win32/gdkhdataoutputstream-win32.c gtk-3.96.0/gdk/win32/gdkhdataoutputstream-win32.c
--- gtk-3.96.0/gdk/win32/gdkhdataoutputstream-win32.c	2019-05-07 06:27:29.000000000 +0300
+++ gtk-3.96.0/gdk/win32/gdkhdataoutputstream-win32.c	2019-07-19 16:10:23.881590461 +0300
@@ -20,7 +20,7 @@
 
 #include "config.h"
 
-#include <Windows.h>
+#include <windows.h>
 
 #include "gdkprivate-win32.h"
 #include "gdkhdataoutputstream-win32.h"
diff -Nurd gtk-3.96.0/gdk/win32/gdkwin32cursor.h gtk-3.96.0/gdk/win32/gdkwin32cursor.h
--- gtk-3.96.0/gdk/win32/gdkwin32cursor.h	2019-07-19 16:09:13.885844010 +0300
+++ gtk-3.96.0/gdk/win32/gdkwin32cursor.h	2019-07-19 16:08:55.649908787 +0300
@@ -29,7 +29,7 @@
 #error "Only <gdk/gdkwin32.h> can be included directly."
 #endif
 
-#include <Windows.h>
+#include <windows.h>
 #include <gdk/gdk.h>
 #include <gdk/win32/gdkwin32display.h>
 
