diff -Nurd icu/source/common/putil.cpp icu/source/common/putil.cpp
--- icu/source/common/putil.cpp	2017-05-24 01:04:43.578563005 +0300
+++ icu/source/common/putil.cpp	2017-05-24 01:41:16.109323731 +0300
@@ -101,7 +101,7 @@
 #   define NOIME
 #   define NOMCX
 #   include <windows.h>
-#   include "unicode\uloc.h"
+#   include "unicode/uloc.h"
 #if U_PLATFORM_HAS_WINUWP_API == 0
 #   include "wintz.h"
 #else // U_PLATFORM_HAS_WINUWP_API
