diff -Nurd icu/source/io/ufile.cpp icu/source/io/ufile.cpp
--- icu/source/io/ufile.cpp	2017-02-19 23:58:30.000000000 +0200
+++ icu/source/io/ufile.cpp	2017-05-24 02:08:48.308333248 +0300
@@ -160,7 +160,7 @@
         // TODO: test this code path, including wperm.
         wchar_t wperm[40] = {};
         size_t  retVal;
-        mbstowcs_s(&retVal, wperm, perm, _TRUNCATE);
+        mbstowcs_s(&retVal, wperm, 40, perm, _TRUNCATE);
         FILE *systemFile = _wfopen((const wchar_t *)filename, wperm);
         if (systemFile) {
             result = finit_owner(systemFile, locale, codepage, TRUE);
