diff -Nurd openal-soft-1.20.1/CMakeLists.txt openal-soft-1.20.1/CMakeLists.txt
--- openal-soft-1.20.1/CMakeLists.txt	2020-01-24 09:28:45.000000000 +0200
+++ openal-soft-1.20.1/CMakeLists.txt	2022-04-07 07:02:47.314470293 +0300
@@ -81,7 +81,7 @@
 SET(EXTRA_LIBS )
 
 IF(WIN32)
-    SET(CPP_DEFS ${CPP_DEFS} _WIN32 _WIN32_WINNT=0x0502)
+    SET(CPP_DEFS ${CPP_DEFS} _WIN32)
 
     OPTION(ALSOFT_BUILD_ROUTER  "Build the router (EXPERIMENTAL; creates OpenAL32.dll and soft_oal.dll)"  OFF)
 
@@ -863,7 +863,6 @@
     ENDIF()
 
     SET(OLD_REQUIRED_DEFINITIONS ${CMAKE_REQUIRED_DEFINITIONS})
-    SET(CMAKE_REQUIRED_DEFINITIONS ${CMAKE_REQUIRED_DEFINITIONS} -D_WIN32_WINNT=0x0502)
 
     # Check MMSystem backend
     CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H)
