diff -Nurd openal-soft-1.19.1/CMakeLists.txt openal-soft-1.19.1/CMakeLists.txt
--- openal-soft-1.19.1/CMakeLists.txt	2018-10-12 01:05:31.000000000 +0300
+++ openal-soft-1.19.1/CMakeLists.txt	2022-03-28 19:20:16.754685656 +0300
@@ -69,7 +69,7 @@
 SET(EXTRA_LIBS )
 
 IF(WIN32)
-    SET(CPP_DEFS ${CPP_DEFS} _WIN32 _WIN32_WINNT=0x0502)
+    SET(CPP_DEFS ${CPP_DEFS} _WIN32)
 
     OPTION(ALSOFT_BUILD_ROUTER  "Build the router (EXPERIMENTAL; creates OpenAL32.dll and soft_oal.dll)"  OFF)
 
@@ -621,7 +621,6 @@
 
 # Check if we have Windows headers
 SET(OLD_REQUIRED_DEFINITIONS ${CMAKE_REQUIRED_DEFINITIONS})
-SET(CMAKE_REQUIRED_DEFINITIONS ${CMAKE_REQUIRED_DEFINITIONS} -D_WIN32_WINNT=0x0502)
 CHECK_INCLUDE_FILE(windows.h HAVE_WINDOWS_H)
 SET(CMAKE_REQUIRED_DEFINITIONS ${OLD_REQUIRED_DEFINITIONS})
 UNSET(OLD_REQUIRED_DEFINITIONS)
@@ -727,8 +726,7 @@
 CHECK_INCLUDE_FILE(stdint.h HAVE_STDINT_H)
 IF(NOT HAVE_STDINT_H)
     IF(HAVE_WINDOWS_H)
-        CHECK_C_SOURCE_COMPILES("#define _WIN32_WINNT 0x0502
-                                 #include <windows.h>
+        CHECK_C_SOURCE_COMPILES("#include <windows.h>
                                  __int64 foo;
                                  int main() {return 0;}" HAVE___INT64)
     ENDIF()
@@ -1074,7 +1072,6 @@
 OPTION(ALSOFT_REQUIRE_WASAPI "Require WASAPI backend" OFF)
 IF(HAVE_WINDOWS_H)
     SET(OLD_REQUIRED_DEFINITIONS ${CMAKE_REQUIRED_DEFINITIONS})
-    SET(CMAKE_REQUIRED_DEFINITIONS ${CMAKE_REQUIRED_DEFINITIONS} -D_WIN32_WINNT=0x0502)
 
     # Check MMSystem backend
     CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H)
