diff -Nurd libpng-1.4.1/configure.ac libpng-1.4.1/configure.ac
--- libpng-1.4.1/configure.ac	2010-02-25 13:38:33.000000000 +0200
+++ libpng-1.4.1/configure.ac	2010-06-01 11:55:14.000000000 +0300
@@ -82,10 +82,7 @@
 
 if test "$have_ld_version_script" = "yes"; then
     AC_MSG_CHECKING([for symbol prefix])
-    SYMBOL_PREFIX=`echo "PREFIX=__USER_LABEL_PREFIX__" \
-                   | ${CPP-${CC-gcc} -E} - 2>&1 \
-                   | ${EGREP-grep} "^PREFIX=" \
-                   | ${SED-sed} "s:^PREFIX=::"`
+    SYMBOL_PREFIX=
     AC_SUBST(SYMBOL_PREFIX)
     AC_MSG_RESULT($SYMBOL_PREFIX)
 fi
