diff -Nurd qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf
--- qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf	2018-06-13 02:59:44.369242282 +0300
+++ qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf	2018-06-13 03:58:44.298142369 +0300
@@ -12,7 +12,7 @@
 # modifications to g++-win32.conf
 
 QMAKE_CC                = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
-QMAKE_CFLAGS           += -fno-keep-inline-dllexport
+QMAKE_CFLAGS           += -fno-keep-inline-dllexport -I$${DLLSPREFIX}/include
 QMAKE_CFLAGS_WARN_ON   += -Wextra
 
 QMAKE_CXX               = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
@@ -21,6 +21,6 @@
 
 QMAKE_LINK              = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
 QMAKE_LINK_C            = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
-QMAKE_LFLAGS            = -static-libgcc -static-libstdc++
+QMAKE_LFLAGS            = -static-libgcc -static-libstdc++ -L$${DLLSPREFIX}/lib
 
 load(qt_config)
