diff -Nurd qt-everywhere-src-5.11.2/qtbase/mkspecs/win32-g++/qmake.conf qt-everywhere-src-5.11.2/qtbase/mkspecs/win32-g++/qmake.conf
--- qt-everywhere-src-5.11.2/qtbase/mkspecs/win32-g++/qmake.conf	2018-10-11 03:27:24.615467224 +0300
+++ qt-everywhere-src-5.11.2/qtbase/mkspecs/win32-g++/qmake.conf	2018-10-11 03:27:53.591326539 +0300
@@ -12,15 +12,15 @@
 # modifications to g++-win32.conf
 
 QMAKE_CC                = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
-QMAKE_CFLAGS           += -fno-keep-inline-dllexport
+QMAKE_CFLAGS           += -fno-keep-inline-dllexport -I$${DLLSPREFIX}/include
 QMAKE_CFLAGS_WARN_ON   += -Wextra
 
 QMAKE_CXX               = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
-QMAKE_CXXFLAGS         += -fno-keep-inline-dllexport
+QMAKE_CXXFLAGS         += -fno-keep-inline-dllexport -I$${DLLSPREFIX}/include
 QMAKE_CXXFLAGS_WARN_ON  = $$QMAKE_CFLAGS_WARN_ON
 
 QMAKE_LINK              = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
 QMAKE_LINK_C            = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
-QMAKE_LFLAGS            = -static-libgcc -static-libstdc++
+QMAKE_LFLAGS            = -static-libgcc -static-libstdc++ -L$${DLLSPREFIX}/lib
 
 load(qt_config)
