diff -Nurd qt-everywhere-src-5.15.2/qtlocation/src/src.pro qt-everywhere-src-5.15.2/qtlocation/src/src.pro
--- qt-everywhere-src-5.15.2/qtlocation/src/src.pro	2020-10-27 10:02:12.000000000 +0200
+++ qt-everywhere-src-5.15.2/qtlocation/src/src.pro	2020-12-12 07:44:53.200386055 +0200
@@ -14,16 +14,6 @@
 SUBDIRS += positioning
 positioning.depends = clip2tri
 
-qtHaveModule(quick) {
-    SUBDIRS += positioningquick location
-    positioningquick.depends += positioning
-    location.depends += positioningquick clip2tri
-
-    plugins.depends += location
-
-    SUBDIRS += imports
-    imports.depends += positioningquick positioning location
-}
 plugins.depends += positioning
 SUBDIRS += plugins
 
