diff -Nurd qt-everywhere-src-5.10.1/qtbase/mkspecs/win32-g++/qmake.conf qt-everywhere-src-5.10.1/qtbase/mkspecs/win32-g++/qmake.conf
--- qt-everywhere-src-5.10.1/qtbase/mkspecs/win32-g++/qmake.conf	2018-02-15 16:39:09.996558417 +0200
+++ qt-everywhere-src-5.10.1/qtbase/mkspecs/win32-g++/qmake.conf	2018-02-15 16:41:32.759816734 +0200
@@ -24,7 +24,7 @@
 # x86_64 platform similar to 'msvc-desktop.conf' toolchain, because, unlike for MSVC,
 # 'QMAKE_TARGET.arch' is inherently unavailable.
 
-QMAKE_CC                = $${CROSS_COMPILE}gcc
+QMAKE_CC                = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
 QMAKE_LEX               = flex
 QMAKE_LEXFLAGS          =
 QMAKE_YACC              = bison -y
@@ -34,7 +34,7 @@
 
 QMAKE_CFLAGS_SSE2      += -mstackrealign
 
-QMAKE_CXX               = $${CROSS_COMPILE}g++
+QMAKE_CXX               = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
 QMAKE_CXXFLAGS          = $$QMAKE_CFLAGS
 QMAKE_CXXFLAGS_WARN_ON  = $$QMAKE_CFLAGS_WARN_ON
 QMAKE_CXXFLAGS_RTTI_ON  = -frtti
@@ -48,8 +48,9 @@
 QMAKE_RUN_CXX           = $(CXX) -c $(CXXFLAGS) $(INCPATH) -o $obj $src
 QMAKE_RUN_CXX_IMP       = $(CXX) -c $(CXXFLAGS) $(INCPATH) -o $@ $<
 
-QMAKE_LINK              = $${CROSS_COMPILE}g++
-QMAKE_LINK_C            = $${CROSS_COMPILE}gcc
+QMAKE_LINK              = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
+QMAKE_LINK_C            = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
+QMAKE_LFLAGS            = -static-libgcc -static-libstdc++
 QMAKE_LFLAGS_EXCEPTIONS_ON = -mthreads
 QMAKE_LFLAGS_RELEASE    = -Wl,-s
 QMAKE_LFLAGS_CONSOLE    = -Wl,-subsystem,console
