diff -Nurd qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf
--- qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf	2018-05-14 07:46:28.000000000 +0300
+++ qt-everywhere-src-5.11.0/qtbase/mkspecs/win32-g++/qmake.conf	2018-06-13 02:21:55.534379257 +0300
@@ -11,15 +11,16 @@
 
 # modifications to g++-win32.conf
 
-QMAKE_CC                = $${CROSS_COMPILE}gcc
+QMAKE_CC                = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
 QMAKE_CFLAGS           += -fno-keep-inline-dllexport
 QMAKE_CFLAGS_WARN_ON   += -Wextra
 
-QMAKE_CXX               = $${CROSS_COMPILE}g++
+QMAKE_CXX               = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
 QMAKE_CXXFLAGS         += -fno-keep-inline-dllexport
 QMAKE_CXXFLAGS_WARN_ON  = $$QMAKE_CFLAGS_WARN_ON
 
-QMAKE_LINK              = $${CROSS_COMPILE}g++
-QMAKE_LINK_C            = $${CROSS_COMPILE}gcc
+QMAKE_LINK              = $${CROSS_COMPILE}g++ -static-libgcc -static-libstdc++
+QMAKE_LINK_C            = $${CROSS_COMPILE}gcc -static-libgcc -static-libstdc++
+QMAKE_LFLAGS            = -static-libgcc -static-libstdc++
 
 load(qt_config)
