diff -Nurd qt-everywhere-src-5.15.2/qtbase/src/corelib/global/qfloat16.h qt-everywhere-src-5.15.2/qtbase/src/corelib/global/qfloat16.h
--- qt-everywhere-src-5.15.2/qtbase/src/corelib/global/qfloat16.h	2021-11-21 14:52:43.075495578 +0200
+++ qt-everywhere-src-5.15.2/qtbase/src/corelib/global/qfloat16.h	2021-11-21 14:52:26.163341118 +0200
@@ -44,6 +44,7 @@
 #include <QtCore/qglobal.h>
 #include <QtCore/qmetatype.h>
 #include <string.h>
+#include <limits>
 
 #if defined(QT_COMPILER_SUPPORTS_F16C) && defined(__AVX2__) && !defined(__F16C__)
 // All processors that support AVX2 do support F16C too. That doesn't mean
diff -Nurd qt-everywhere-src-5.15.2/qtbase/src/corelib/text/qbytearraymatcher.h qt-everywhere-src-5.15.2/qtbase/src/corelib/text/qbytearraymatcher.h
--- qt-everywhere-src-5.15.2/qtbase/src/corelib/text/qbytearraymatcher.h	2021-11-21 14:52:43.075495578 +0200
+++ qt-everywhere-src-5.15.2/qtbase/src/corelib/text/qbytearraymatcher.h	2021-11-21 14:52:26.187341338 +0200
@@ -40,6 +40,8 @@
 #ifndef QBYTEARRAYMATCHER_H
 #define QBYTEARRAYMATCHER_H
 
+#include <limits>
+
 #include <QtCore/qbytearray.h>
 
 QT_BEGIN_NAMESPACE
diff -Nurd qt-everywhere-src-5.15.2/qtbase/src/corelib/tools/qoffsetstringarray_p.h qt-everywhere-src-5.15.2/qtbase/src/corelib/tools/qoffsetstringarray_p.h
--- qt-everywhere-src-5.15.2/qtbase/src/corelib/tools/qoffsetstringarray_p.h	2020-10-27 10:02:11.000000000 +0200
+++ qt-everywhere-src-5.15.2/qtbase/src/corelib/tools/qoffsetstringarray_p.h	2021-11-21 14:53:31.419934387 +0200
@@ -55,6 +55,7 @@
 
 #include <tuple>
 #include <array>
+#include <limits>
 
 QT_BEGIN_NAMESPACE
 
