diff -Nurd qt-everywhere-src-5.15.2/qtbase/src/plugins/platforms/windows/qwindowsservices.cpp qt-everywhere-src-5.15.2/qtbase/src/plugins/platforms/windows/qwindowsservices.cpp
--- qt-everywhere-src-5.15.2/qtbase/src/plugins/platforms/windows/qwindowsservices.cpp	2020-10-27 10:02:11.000000000 +0200
+++ qt-everywhere-src-5.15.2/qtbase/src/plugins/platforms/windows/qwindowsservices.cpp	2020-12-12 05:12:20.910457389 +0200
@@ -56,6 +56,7 @@
 
 enum { debug = 0 };
 
+#if 0
 class QWindowsShellExecuteThread : public QThread
 {
 public:
@@ -75,6 +76,7 @@
     HINSTANCE m_result = nullptr;
     const wchar_t *m_path;
 };
+#endif
 
 static inline bool shellExecute(const QUrl &url)
 {
@@ -83,6 +85,7 @@
         : url.toString(QUrl::FullyEncoded);
 
 
+#if 0
     // Run ShellExecute() in a thread since it may spin the event loop.
     // Prevent it from interfering with processing of posted events (QTBUG-85676).
     QWindowsShellExecuteThread thread(reinterpret_cast<const wchar_t *>(nativeFilePath.utf16()));
@@ -90,6 +93,12 @@
     thread.wait();
 
     const auto result = reinterpret_cast<quintptr>(thread.result());
+#endif
+
+    const auto result =
+        reinterpret_cast<quintptr>(ShellExecute(nullptr, nullptr,
+                                                reinterpret_cast<const wchar_t *>(nativeFilePath.utf16()),
+                                                nullptr, nullptr, SW_SHOWNORMAL));
 
     // ShellExecute returns a value greater than 32 if successful
     if (result <= 32) {
