diff -Nurd qt-everywhere-src-5.15.2/qtbase/mkspecs/features/uikit/devices.py qt-everywhere-src-5.15.2/qtbase/mkspecs/features/uikit/devices.py
--- qt-everywhere-src-5.15.2/qtbase/mkspecs/features/uikit/devices.py	2020-10-27 10:02:11.000000000 +0200
+++ qt-everywhere-src-5.15.2/qtbase/mkspecs/features/uikit/devices.py	2020-12-10 23:30:59.544183336 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 #############################################################################
 ##
diff -Nurd qt-everywhere-src-5.15.2/qtscxml/tests/3rdparty/gen-scion-tests.py qt-everywhere-src-5.15.2/qtscxml/tests/3rdparty/gen-scion-tests.py
--- qt-everywhere-src-5.15.2/qtscxml/tests/3rdparty/gen-scion-tests.py	2020-10-15 09:53:19.000000000 +0300
+++ qt-everywhere-src-5.15.2/qtscxml/tests/3rdparty/gen-scion-tests.py	2020-12-10 23:38:09.807503196 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 # Copyright (C) 2016 The Qt Company Ltd.
 # Contact: https://www.qt.io/licensing/
