diff -Nurd qt-everywhere-src-5.12.6/qtbase/mkspecs/features/uikit/devices.py qt-everywhere-src-5.12.6/qtbase/mkspecs/features/uikit/devices.py
--- qt-everywhere-src-5.12.6/qtbase/mkspecs/features/uikit/devices.py	2019-12-02 06:06:26.822719528 +0200
+++ qt-everywhere-src-5.12.6/qtbase/mkspecs/features/uikit/devices.py	2019-12-02 05:59:31.820816119 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 #############################################################################
 ##
diff -Nurd qt-everywhere-src-5.12.6/qtdeclarative/qtdeclarative.pro qt-everywhere-src-5.12.6/qtdeclarative/qtdeclarative.pro
--- qt-everywhere-src-5.12.6/qtdeclarative/qtdeclarative.pro	2019-12-02 06:06:26.862719698 +0200
+++ qt-everywhere-src-5.12.6/qtdeclarative/qtdeclarative.pro	2019-12-02 06:00:19.293052188 +0200
@@ -2,8 +2,8 @@
 load(qt_parts)
 
 !python_available {
-    py_out = $$system('python -c "print(1)"')
-    !equals(py_out, 1): error("Building QtQml requires Python.")
+    py_out = $$system('python3 -c "print(1)"')
+    !equals(py_out, 1): error("Building QtQml requires Python3.")
     tmp = python_available
     CONFIG += $$tmp
     cache(CONFIG, add, tmp)
diff -Nurd qt-everywhere-src-5.12.6/qtdeclarative/src/3rdparty/masm/masm.pri qt-everywhere-src-5.12.6/qtdeclarative/src/3rdparty/masm/masm.pri
--- qt-everywhere-src-5.12.6/qtdeclarative/src/3rdparty/masm/masm.pri	2019-11-07 13:01:48.000000000 +0200
+++ qt-everywhere-src-5.12.6/qtdeclarative/src/3rdparty/masm/masm.pri	2019-12-02 06:09:35.743501251 +0200
@@ -58,7 +58,7 @@
     udis86.output = udis86_itab.h
     udis86.input = ITAB
     udis86.CONFIG += no_link
-    udis86.commands = python $$PWD/disassembler/udis86/itab.py ${QMAKE_FILE_IN}
+    udis86.commands = python3 $$PWD/disassembler/udis86/itab.py ${QMAKE_FILE_IN}
     QMAKE_EXTRA_COMPILERS += udis86
 
     udis86_tab_cfile.target = $$OUT_PWD/udis86_itab.c
@@ -113,7 +113,7 @@
 retgen.script = $$PWD/yarr/create_regex_tables
 retgen.input = retgen.script
 retgen.CONFIG += no_link
-retgen.commands = python $$retgen.script > ${QMAKE_FILE_OUT}
+retgen.commands = python3 $$retgen.script > ${QMAKE_FILE_OUT}
 QMAKE_EXTRA_COMPILERS += retgen
 
 # Taken from WebKit/Tools/qmake/mkspecs/features/unix/default_post.prf
diff -Nurd qt-everywhere-src-5.12.6/qtwebengine/src/3rdparty/chromium/third_party/pywebsocket/src/setup.py qt-everywhere-src-5.12.6/qtwebengine/src/3rdparty/chromium/third_party/pywebsocket/src/setup.py
--- qt-everywhere-src-5.12.6/qtwebengine/src/3rdparty/chromium/third_party/pywebsocket/src/setup.py	2019-12-02 06:06:26.878719765 +0200
+++ qt-everywhere-src-5.12.6/qtwebengine/src/3rdparty/chromium/third_party/pywebsocket/src/setup.py	2019-12-02 06:05:31.690483713 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2012, Google Inc.
 # All rights reserved.
