diff -Nurd qt-everywhere-src-5.15.2/qtbase/src/src.pro qt-everywhere-src-5.15.2/qtbase/src/src.pro
--- qt-everywhere-src-5.15.2/qtbase/src/src.pro	2020-10-27 10:02:11.000000000 +0200
+++ qt-everywhere-src-5.15.2/qtbase/src/src.pro	2020-12-11 01:36:40.358781629 +0200
@@ -175,7 +175,6 @@
     src_plugins.depends += src_sql
 }
 qtConfig(xml): SUBDIRS += src_xml
-qtConfig(testlib): SUBDIRS += src_testlib
 
 qtConfig(dbus) {
     force_dbus_bootstrap|qtConfig(private_tests): \
@@ -258,13 +257,6 @@
         SUBDIRS += src_sql_doc_snippets
     }
 
-    qtConfig(testlib):qtConfig(widgets):qtConfig(sql) {
-        src_testlib_doc_snippets.subdir = testlib/doc/snippets
-        src_testlib_doc_snippets.target = sub-testlib-doc-snippets
-        src_testlib_doc_snippets.depends = src_testlib src_widgets src_sql
-        SUBDIRS += src_testlib_doc_snippets
-    }
-
     qtConfig(widgets):qtConfig(printer):qtConfig(opengl) {
         src_widgets_doc_snippets.subdir = widgets/doc/snippets
         src_widgets_doc_snippets.target = sub-widgets-doc-snippets
