diff -Nurd qt-everywhere-src-5.15.1/qtbase/src/testlib/qtestcase.cpp qt-everywhere-src-5.15.1/qtbase/src/testlib/qtestcase.cpp
--- qt-everywhere-src-5.15.1/qtbase/src/testlib/qtestcase.cpp	2020-12-12 02:04:41.398631223 +0200
+++ qt-everywhere-src-5.15.1/qtbase/src/testlib/qtestcase.cpp	2020-12-12 02:06:24.107597412 +0200
@@ -59,7 +59,7 @@
 #include <QtCore/qtemporarydir.h>
 #include <QtCore/qthread.h>
 #include <QtCore/private/qlocking_p.h>
-#include <QtCore/private/qwaitcondition_p.h>
+//#include <QtCore/private/qwaitcondition_p.h>
 
 #include <QtCore/qtestsupport_core.h>
 
@@ -349,7 +349,7 @@
 static int keyDelay = -1;
 static int mouseDelay = -1;
 static int eventDelay = -1;
-#if QT_CONFIG(thread)
+#if 0
 static int timeout = -1;
 #endif
 static bool noCrashHandler = false;
@@ -402,7 +402,7 @@
     }
     return keyDelay;
 }
-#if QT_CONFIG(thread)
+#if 0
 static std::chrono::milliseconds defaultTimeout()
 {
     if (timeout == -1) {
@@ -1001,7 +1001,7 @@
     }
 }
 
-#if QT_CONFIG(thread)
+#if 0
 
 class WatchDog : public QThread
 {
@@ -1082,7 +1082,7 @@
     Expectation expecting;
 };
 
-#else // !QT_CONFIG(thread)
+#else // 0
 
 class WatchDog : public QObject
 {
