diff -Nurd qt-everywhere-src-5.11.0/qtbase/src/plugins/platforms/windows/qwin10helpers.cpp qt-everywhere-src-5.11.0/qtbase/src/plugins/platforms/windows/qwin10helpers.cpp
--- qt-everywhere-src-5.11.0/qtbase/src/plugins/platforms/windows/qwin10helpers.cpp	2018-05-14 07:46:28.000000000 +0300
+++ qt-everywhere-src-5.11.0/qtbase/src/plugins/platforms/windows/qwin10helpers.cpp	2018-06-13 03:01:04.468774618 +0300
@@ -44,7 +44,7 @@
 #include <QtCore/private/qsystemlibrary_p.h>
 
 #if defined(Q_CC_MINGW) || defined(Q_CC_CLANG)
-#  define HAS_UI_VIEW_SETTINGS_INTEROP
+//#  define HAS_UI_VIEW_SETTINGS_INTEROP
 // Present from MSVC2015 + SDK 10 onwards
 #elif (!defined(Q_CC_MSVC) || _MSC_VER >= 1900) && NTDDI_VERSION >= 0xa000000
 #  define HAS_UI_VIEW_SETTINGS_INTEROP
