diff -Nurd qt-everywhere-src-5.14.0/qtbase/qmake/doc/qmake.qdocconf qt-everywhere-src-5.14.0/qtbase/qmake/doc/qmake.qdocconf
--- qt-everywhere-src-5.14.0/qtbase/qmake/doc/qmake.qdocconf	2020-03-07 13:48:38.907471811 +0200
+++ qt-everywhere-src-5.14.0/qtbase/qmake/doc/qmake.qdocconf	2020-03-07 13:53:54.552190851 +0200
@@ -60,7 +60,6 @@
     qttestlib \
     qtuitools \
     qtwidgets \
-    qtwinextras \
     qtx11extras \
     qtxml \
     qtxmlpatterns
diff -Nurd qt-everywhere-src-5.14.0/qtbase/src/corelib/doc/qtcore.qdocconf qt-everywhere-src-5.14.0/qtbase/src/corelib/doc/qtcore.qdocconf
--- qt-everywhere-src-5.14.0/qtbase/src/corelib/doc/qtcore.qdocconf	2020-03-07 13:48:38.615471172 +0200
+++ qt-everywhere-src-5.14.0/qtbase/src/corelib/doc/qtcore.qdocconf	2020-03-07 13:53:54.552190851 +0200
@@ -26,7 +26,7 @@
 
 tagfile                 = ../../../doc/qtcore/qtcore.tags
 
-depends                 += activeqt qtdbus qtgui qtwidgets qtnetwork qtdoc qtmacextras qtquick qtlinguist qtdesigner qtconcurrent qtxml qmake qtwinextras qtqml
+depends                 += activeqt qtdbus qtgui qtwidgets qtnetwork qtdoc qtmacextras qtquick qtlinguist qtdesigner qtconcurrent qtxml qmake qtqml
 
 headerdirs  += ..
 
diff -Nurd qt-everywhere-src-5.14.0/qtwinextras/src/src.pro qt-everywhere-src-5.14.0/qtwinextras/src/src.pro
--- qt-everywhere-src-5.14.0/qtwinextras/src/src.pro	2020-03-07 13:48:43.595482089 +0200
+++ qt-everywhere-src-5.14.0/qtwinextras/src/src.pro	2020-03-07 13:53:54.552190851 +0200
@@ -1,8 +1,6 @@
 win32 {
     TEMPLATE = subdirs
     CONFIG += ordered
-    SUBDIRS += winextras
-    qtHaveModule(quick): SUBDIRS += imports
 } else {
     # fake project for creating the documentation
     TEMPLATE = aux
