diff -Nurd qt-everywhere-src-5.11.2/qtbase/qmake/doc/qmake.qdocconf qt-everywhere-src-5.11.2/qtbase/qmake/doc/qmake.qdocconf
--- qt-everywhere-src-5.11.2/qtbase/qmake/doc/qmake.qdocconf	2018-10-11 23:40:12.406287417 +0300
+++ qt-everywhere-src-5.11.2/qtbase/qmake/doc/qmake.qdocconf	2018-10-11 23:38:49.494728916 +0300
@@ -58,7 +58,6 @@
     qttestlib \
     qtuitools \
     qtwidgets \
-    qtwinextras \
     qtx11extras \
     qtxml \
     qtxmlpatterns
diff -Nurd qt-everywhere-src-5.11.2/qtbase/src/corelib/doc/qtcore.qdocconf qt-everywhere-src-5.11.2/qtbase/src/corelib/doc/qtcore.qdocconf
--- qt-everywhere-src-5.11.2/qtbase/src/corelib/doc/qtcore.qdocconf	2018-10-11 23:40:12.410287396 +0300
+++ qt-everywhere-src-5.11.2/qtbase/src/corelib/doc/qtcore.qdocconf	2018-10-11 23:38:49.702727799 +0300
@@ -25,7 +25,7 @@
 
 tagfile                 = ../../../doc/qtcore/qtcore.tags
 
-depends                 += activeqt qtdbus qtgui qtwidgets qtnetwork qtdoc qtmacextras qtquick qtlinguist qtdesigner qtconcurrent qtxml qmake qtwinextras qtqml
+depends                 += activeqt qtdbus qtgui qtwidgets qtnetwork qtdoc qtmacextras qtquick qtlinguist qtdesigner qtconcurrent qtxml qmake qtqml
 
 headerdirs  += ..
 
diff -Nurd qt-everywhere-src-5.11.2/qtdoc/doc/config/qtdoc.qdocconf qt-everywhere-src-5.11.2/qtdoc/doc/config/qtdoc.qdocconf
--- qt-everywhere-src-5.11.2/qtdoc/doc/config/qtdoc.qdocconf	2018-10-11 23:40:12.410287396 +0300
+++ qt-everywhere-src-5.11.2/qtdoc/doc/config/qtdoc.qdocconf	2018-10-11 23:38:48.594733750 +0300
@@ -55,7 +55,6 @@
     qtwebengine \
     qtwebsockets \
     qtwidgets \
-    qtwinextras \
     qtx11extras \
     qtxml \
     qtxmlpatterns \
diff -Nurd qt-everywhere-src-5.11.2/qtwinextras/src/src.pro qt-everywhere-src-5.11.2/qtwinextras/src/src.pro
--- qt-everywhere-src-5.11.2/qtwinextras/src/src.pro	2018-10-11 23:40:12.410287396 +0300
+++ qt-everywhere-src-5.11.2/qtwinextras/src/src.pro	2018-10-11 23:42:44.877492367 +0300
@@ -1,8 +1,6 @@
 win32 {
     TEMPLATE = subdirs
     CONFIG += ordered
-    SUBDIRS += winextras
-    qtHaveModule(quick): SUBDIRS += imports
 } else {
     # fake project for creating the documentation
     TEMPLATE = aux
