diff -Nurd qt-everywhere-src-5.14.2/qtbase/qmake/doc/qmake.qdocconf qt-everywhere-src-5.14.2/qtbase/qmake/doc/qmake.qdocconf
--- qt-everywhere-src-5.14.2/qtbase/qmake/doc/qmake.qdocconf	2020-03-27 11:49:31.000000000 +0200
+++ qt-everywhere-src-5.14.2/qtbase/qmake/doc/qmake.qdocconf	2020-04-02 01:15:35.991109638 +0300
@@ -60,7 +60,6 @@
     qttestlib \
     qtuitools \
     qtwidgets \
-    qtwinextras \
     qtx11extras \
     qtxml \
     qtxmlpatterns
diff -Nurd qt-everywhere-src-5.14.2/qtbase/src/corelib/doc/qtcore.qdocconf qt-everywhere-src-5.14.2/qtbase/src/corelib/doc/qtcore.qdocconf
--- qt-everywhere-src-5.14.2/qtbase/src/corelib/doc/qtcore.qdocconf	2020-03-27 11:49:31.000000000 +0200
+++ qt-everywhere-src-5.14.2/qtbase/src/corelib/doc/qtcore.qdocconf	2020-04-02 01:16:30.843442112 +0300
@@ -26,7 +26,7 @@
 
 tagfile                 = ../../../doc/qtcore/qtcore.tags
 
-depends                 += activeqt qtdbus qtgui qtwidgets qtnetwork qtdoc qtmacextras qtquick qtlinguist qtdesigner qtconcurrent qtxml qmake qtwinextras qtqml qtcmake
+depends                 += activeqt qtdbus qtgui qtwidgets qtnetwork qtdoc qtmacextras qtquick qtlinguist qtdesigner qtconcurrent qtxml qmake qtqml qtcmake
 
 headerdirs  += ..
 
diff -Nurd qt-everywhere-src-5.14.2/qtwinextras/src/src.pro qt-everywhere-src-5.14.2/qtwinextras/src/src.pro
--- qt-everywhere-src-5.14.2/qtwinextras/src/src.pro	2020-03-03 15:10:32.000000000 +0200
+++ qt-everywhere-src-5.14.2/qtwinextras/src/src.pro	2020-04-02 01:15:35.991109638 +0300
@@ -1,8 +1,6 @@
 win32 {
     TEMPLATE = subdirs
     CONFIG += ordered
-    SUBDIRS += winextras
-    qtHaveModule(quick): SUBDIRS += imports
 } else {
     # fake project for creating the documentation
     TEMPLATE = aux
