diff -Nurd qt-everywhere-src-5.13.0/qtdeclarative/src/3rdparty/masm/yarr/YarrCanonicalize.h qt-everywhere-src-5.13.0/qtdeclarative/src/3rdparty/masm/yarr/YarrCanonicalize.h
--- qt-everywhere-src-5.13.0/qtdeclarative/src/3rdparty/masm/yarr/YarrCanonicalize.h	2019-05-09 07:28:41.000000000 +0300
+++ qt-everywhere-src-5.13.0/qtdeclarative/src/3rdparty/masm/yarr/YarrCanonicalize.h	2019-06-23 14:08:34.188295312 +0300
@@ -26,7 +26,7 @@
 #pragma once
 
 #include <stdint.h>
-#include <unicode/utypes.h>
+#include <wtf/unicode/utypes.h>
 
 namespace JSC { namespace Yarr {
 
