diff -Nurd readline-6.3/input.c readline-6.3/input.c
--- readline-6.3/input.c	2014-01-10 22:07:08.000000000 +0200
+++ readline-6.3/input.c	2015-06-24 00:42:45.685167551 +0300
@@ -532,9 +532,17 @@
 	 Otherwise (not EINTR), some error occurred, also signifying EOF. */
       if (errno != EINTR)
 	return (RL_ISSTATE (RL_STATE_READCMD) ? READERR : EOF);
-      else if (_rl_caught_signal == SIGHUP || _rl_caught_signal == SIGTERM)
+      else if (
+#if defined (SIGHUP)
+	       _rl_caught_signal == SIGHUP ||
+#endif
+	       _rl_caught_signal == SIGTERM)
 	return (RL_ISSTATE (RL_STATE_READCMD) ? READERR : EOF);
-      else if (_rl_caught_signal == SIGINT || _rl_caught_signal == SIGQUIT)
+      else if (_rl_caught_signal == SIGINT
+#if defined (SIGQUIT)
+	       || _rl_caught_signal == SIGQUIT
+#endif
+	       )
         RL_CHECK_SIGNALS ();
 
       if (rl_signal_event_hook)
diff -Nurd readline-6.3/signals.c readline-6.3/signals.c
--- readline-6.3/signals.c	2015-06-24 00:41:34.181169314 +0300
+++ readline-6.3/signals.c	2015-06-24 00:41:22.741169596 +0300
@@ -216,7 +216,9 @@
       /* FALLTHROUGH */
 
     case SIGTERM:
+#if defined (SIGHUP)
     case SIGHUP:
+#endif
 #if defined (SIGTSTP)
     case SIGTSTP:
     case SIGTTOU:
@@ -397,7 +399,9 @@
 
       sigaddset (&bset, SIGINT);
       sigaddset (&bset, SIGTERM);
+#if defined (SIGHUP)
       sigaddset (&bset, SIGHUP);
+#endif
 #if defined (SIGQUIT)
       sigaddset (&bset, SIGQUIT);
 #endif
@@ -426,7 +430,9 @@
 
       rl_maybe_set_sighandler (SIGINT, rl_signal_handler, &old_int);
       rl_maybe_set_sighandler (SIGTERM, rl_signal_handler, &old_term);
+#if defined (SIGHUP)
       rl_maybe_set_sighandler (SIGHUP, rl_signal_handler, &old_hup);
+#endif
 #if defined (SIGQUIT)
       rl_maybe_set_sighandler (SIGQUIT, rl_signal_handler, &old_quit);
 #endif
@@ -491,7 +497,9 @@
 	 overhead */
       rl_maybe_restore_sighandler (SIGINT, &old_int);
       rl_maybe_restore_sighandler (SIGTERM, &old_term);
+#if defined (SIGHUP)
       rl_maybe_restore_sighandler (SIGHUP, &old_hup);
+#endif
 #if defined (SIGQUIT)
       rl_maybe_restore_sighandler (SIGQUIT, &old_quit);
 #endif
