diff -Nurd shared-mime-info-2.2/data/meson.build shared-mime-info-2.2/data/meson.build
--- shared-mime-info-2.2/data/meson.build	2022-03-27 13:19:00.000000000 +0300
+++ shared-mime-info-2.2/data/meson.build	2022-08-18 09:24:44.563714383 +0300
@@ -16,7 +16,7 @@
   install_dir : get_option('datadir') / 'gettext/its'
 )
 
-if xmlto.found()
+if xmlto.found() and get_option('build-spec-html')
     custom_target('shared-mime-info-spec-html',
         input : 'shared-mime-info-spec.xml',
         output: 'shared-mime-info-spec-html',
diff -Nurd shared-mime-info-2.2/meson_options.txt shared-mime-info-2.2/meson_options.txt
--- shared-mime-info-2.2/meson_options.txt	2022-03-27 13:19:00.000000000 +0300
+++ shared-mime-info-2.2/meson_options.txt	2022-08-18 09:25:58.836800355 +0300
@@ -10,3 +10,7 @@
 option('xdgmime-path', type: 'string', value: './xdgmime',
     description: 'Path to the xdgmime executable',
 )
+
+option('build-spec-html', type: 'boolean', value: true,
+    description: 'Build the shared-mime-info-spec-html. Requires xmlto.',
+)
