diff -Nurd svgalib-1.9.25/Makefile.cfg svgalib-1.9.25/Makefile.cfg
--- svgalib-1.9.25/Makefile.cfg	2009-05-26 14:36:01.000000000 +0300
+++ svgalib-1.9.25/Makefile.cfg	2009-05-26 14:43:54.000000000 +0300
@@ -30,7 +30,7 @@
 INCLUDEDIR = /usr/src/linux/include
 endif
 
-ARCH := $(shell uname -m | sed -e s/i.86/i386/ -e s/sun4u/sparc64/ -e s/arm.*/arm/ -e s/sa110/arm/)
+ARCH := <ARCH>
 
 #----------------------------------------------------------------------
 # Configuration Section
