diff -Nurd tango-icon-theme-0.8.90/configure tango-icon-theme-0.8.90/configure
--- tango-icon-theme-0.8.90/configure	2009-02-26 05:08:00.000000000 +0200
+++ tango-icon-theme-0.8.90/configure	2023-07-19 12:57:42.158152198 +0300
@@ -6410,9 +6410,9 @@
 
 { echo "$as_me:$LINENO: checking icon-naming-utils >= $UTILS_REQUIRED" >&5
 echo $ECHO_N "checking icon-naming-utils >= $UTILS_REQUIRED... $ECHO_C" >&6; }
-if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"icon-naming-utils >= \$UTILS_REQUIRED\"") >&5
-  ($PKG_CONFIG --exists --print-errors "icon-naming-utils >= $UTILS_REQUIRED") 2>&5
+if test -n "${PKG_CONFIG_FOR_BUILD}" && \
+    { (echo "$as_me:$LINENO: \${PKG_CONFIG_FOR_BUILD} --exists --print-errors \"icon-naming-utils >= \$UTILS_REQUIRED\"") >&5
+  (PKG_CONFIG_PATH="${PKG_PATH_FOR_BUILD}" "${PKG_CONFIG_FOR_BUILD}" --exists --print-errors "icon-naming-utils >= $UTILS_REQUIRED") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
@@ -6421,7 +6421,7 @@
   have_utils=no
 fi
 if test "x$have_utils" = "xyes"; then
-   UTILS_PATH="`$PKG_CONFIG --variable=program_path icon-naming-utils`"
+   UTILS_PATH="$(PKG_CONFIG_PATH="${PKG_PATH_FOR_BUILD}" "${PKG_CONFIG_FOR_BUILD}" --variable=program_path icon-naming-utils)"
    ICONMAP="$UTILS_PATH/icon-name-mapping"
 
    { echo "$as_me:$LINENO: result: yes" >&5
@@ -6454,17 +6454,17 @@
 
 
 if test "x$enable_framing" = "xyes"; then
-   if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"ImageMagick >= 5.5.7\"") >&5
-  ($PKG_CONFIG --exists --print-errors "ImageMagick >= 5.5.7") 2>&5
+   if test -n "${PKG_CONFIG_FOR_BUILD}" && \
+    { (echo "$as_me:$LINENO: \${PKG_CONFIG_FOR_BUILD} --exists --print-errors \"ImageMagick >= 5.5.7\"") >&5
+  (PKG_CONFIG_PATH="${PKG_PATH_FOR_BUILD}" "${PKG_CONFIG_FOR_BUILD}" --exists --print-errors "ImageMagick >= 5.5.7") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
   enable_framing=yes
 else
-  if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"GraphicsMagick >= 1.1\"") >&5
-  ($PKG_CONFIG --exists --print-errors "GraphicsMagick >= 1.1") 2>&5
+  if test -n "${PKG_CONFIG_FOR_BUILD}" && \
+    { (echo "$as_me:$LINENO: \${PKG_CONFIG_FOR_BUILD} --exists --print-errors \"GraphicsMagick >= 1.1\"") >&5
+  (PKG_CONFIG_PATH="${PKG_PATH_FOR_BUILD}" "${PKG_CONFIG_FOR_BUILD}" --exists --print-errors "GraphicsMagick >= 1.1") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
@@ -6543,9 +6543,9 @@
 { echo "$as_me:$LINENO: checking SVG to PNG conversion method" >&5
 echo $ECHO_N "checking SVG to PNG conversion method... $ECHO_C" >&6; }
 if test "x$enable_large_bitmaps" = "xyes"; then
-   if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"librsvg-2.0 >= 2.12.3\"") >&5
-  ($PKG_CONFIG --exists --print-errors "librsvg-2.0 >= 2.12.3") 2>&5
+   if test -n "${PKG_CONFIG_FOR_BUILD}" && \
+    { (echo "$as_me:$LINENO: \${PKG_CONFIG_FOR_BUILD} --exists --print-errors \"librsvg-2.0 >= 2.12.3\"") >&5
+  (PKG_CONFIG_PATH="${PKG_PATH_FOR_BUILD}" "${PKG_CONFIG_FOR_BUILD}" --exists --print-errors "librsvg-2.0 >= 2.12.3") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
