diff -Nurd libxkbcommon-0.8.4/src/atom.c libxkbcommon-0.8.4/src/atom.c
--- libxkbcommon-0.8.4/src/atom.c	2019-12-01 16:34:50.026942389 +0200
+++ libxkbcommon-0.8.4/src/atom.c	2019-12-01 16:39:11.437982973 +0200
@@ -73,6 +73,25 @@
 #include "utils.h"
 #include "atom.h"
 
+char *strndup(const char *s, size_t n);
+
+char *strndup(const char *s, size_t n)
+{
+  int len = strlen(s);
+  char *ret;
+
+  if (n < len) {
+    len = n;
+  }
+  ret = malloc(len + 1);
+  if (ret != NULL) {
+    memcpy(ret, s, len);
+    ret[len] = '\0';
+  }
+
+  return ret;
+}
+
 struct atom_node {
     xkb_atom_t left, right;
     xkb_atom_t atom;
diff -Nurd libxkbcommon-0.8.4/src/compose/paths.c libxkbcommon-0.8.4/src/compose/paths.c
--- libxkbcommon-0.8.4/src/compose/paths.c	2019-12-01 16:34:50.026942389 +0200
+++ libxkbcommon-0.8.4/src/compose/paths.c	2019-12-01 16:34:23.151043372 +0200
@@ -24,6 +24,8 @@
 #include "utils.h"
 #include "paths.h"
 
+char *strndup(const char *s, size_t n);
+
 enum resolve_name_direction {
     LEFT_TO_RIGHT,
     RIGHT_TO_LEFT,
