diff -Nurd eglibc/Makeconfig eglibc/Makeconfig
--- eglibc/Makeconfig	2009-11-09 14:42:08.000000000 +0200
+++ eglibc/Makeconfig	2009-11-09 14:43:44.000000000 +0200
@@ -943,7 +943,7 @@
 all-subdirs = csu assert ctype locale intl catgets math setjmp signal	    \
 	      stdlib stdio-common libio malloc string wcsmbs time dirent    \
 	      grp pwd posix io termios resource misc socket sysvipc gmon    \
-	      gnulib iconv iconvdata wctype manual shadow gshadow po argp   \
+	      gnulib iconv iconvdata wctype shadow gshadow po argp          \
 	      crypt nss localedata timezone rt conform debug		    \
 	      $(add-on-subdirs) $(dlfcn) $(binfmt-subdir)
 
diff -Nurd eglibc/sysdeps/gnu/Makefile eglibc/sysdeps/gnu/Makefile
--- eglibc/sysdeps/gnu/Makefile	2009-11-09 14:42:08.000000000 +0200
+++ eglibc/sysdeps/gnu/Makefile	2009-11-09 14:42:50.000000000 +0200
@@ -20,8 +20,7 @@
 # Generate the list of strings for errno codes from the section of the
 # manual which documents the codes.
 
-$(..)sysdeps/gnu/errlist.c: $(..)sysdeps/gnu/errlist.awk \
-			    $(..)manual/errno.texi
+$(..)sysdeps/gnu/errlist.c: $(..)sysdeps/gnu/errlist.awk
 	$(AWK) -f $^ > $@-tmp
 # Make it unwritable so noone will edit it by mistake.
 	-chmod a-w $@-tmp
