diff -Nurd jpeg-6b/configure jpeg-6b/configure
--- jpeg-6b/configure	2008-05-02 05:43:31.000000000 +0300
+++ jpeg-6b/configure	2008-05-02 12:37:42.000000000 +0300
@@ -1510,6 +1510,34 @@
   echo "$ac_t""no" 1>&6
 fi
 
+# Extract the first word of "ar", so it can be a program name with args.
+set dummy ar; ac_word=$2
+echo $ac_n "checking for $ac_word""... $ac_c" 1>&6
+echo "configure:1488: checking for $ac_word" >&5
+if eval "test \"`echo '$''{'ac_cv_prog_AR'+set}'`\" = set"; then
+  echo $ac_n "(cached) $ac_c" 1>&6
+else
+  if test -n "$AR"; then
+  ac_cv_prog_AR="$AR" # Let the user override the test.
+else
+  IFS="${IFS= 	}"; ac_save_ifs="$IFS"; IFS="${IFS}:"
+  for ac_dir in $PATH; do
+    test -z "$ac_dir" && ac_dir=.
+    if test -f $ac_dir/$ac_word; then
+      ac_cv_prog_AR="ar"
+      break
+    fi
+  done
+  IFS="$ac_save_ifs"
+  test -z "$ac_cv_prog_AR" && ac_cv_prog_AR=":"
+fi
+fi
+AR="$ac_cv_prog_AR"
+if test -n "$AR"; then
+  echo "$ac_t""$AR" 1>&6
+else
+  echo "$ac_t""no" 1>&6
+fi
 
 # Decide whether to use libtool,
 # and if so whether to build shared, static, or both flavors of library.
@@ -1783,6 +1811,7 @@
 s%@INSTALL_PROGRAM@%$INSTALL_PROGRAM%g
 s%@INSTALL_DATA@%$INSTALL_DATA%g
 s%@RANLIB@%$RANLIB%g
+s%@AR@%$AR%g
 s%@LIBTOOL@%$LIBTOOL%g
 s%@LIBTOOL_CC_TAG@%$LIBTOOL_CC_TAG%g
 s%@O@%$O%g
diff -Nurd jpeg-6b/makefile.cfg jpeg-6b/makefile.cfg
--- jpeg-6b/makefile.cfg	2008-05-02 05:43:31.000000000 +0300
+++ jpeg-6b/makefile.cfg	2008-05-02 12:35:32.000000000 +0300
@@ -62,7 +62,7 @@
 # directory creation command
 MKDIR= mkdir
 # library (.a) file creation command
-AR= ar rc
+AR= @AR@ rc
 # second step in .a creation (use "touch" if not needed)
 AR2= @RANLIB@
 # installation program
