diff -Nurd gtk+-3.8.0/gtk/native/Makefile.am gtk+-3.8.0/gtk/native/Makefile.am
--- gtk+-3.8.0/gtk/native/Makefile.am	2013-03-09 22:03:21.000000000 +0200
+++ gtk+-3.8.0/gtk/native/Makefile.am	2013-04-05 02:56:16.808668461 +0300
@@ -7,7 +7,7 @@
 if CROSS_COMPILING
 if !USE_EXTERNAL_ICON_CACHE
 noinst_PROGRAMS = native-update-icon-cache
-native_update_icon_cache_CFLAGS = $(NATIVE_GDKPIXBUF_CFLAGS)
+native_update_icon_cache_CFLAGS = $(NATIVE_GDKPIXBUF_CFLAGS) -DNATIVE_BUILD
 native_update_icon_cache_LDADD = $(NATIVE_GDKPIXBUF_LIBS)
 native_update_icon_cache_SOURCES = $(srcdir)/../updateiconcache.c
 endif
diff -Nurd gtk+-3.8.0/gtk/native/Makefile.in gtk+-3.8.0/gtk/native/Makefile.in
--- gtk+-3.8.0/gtk/native/Makefile.in	2013-03-25 22:59:36.000000000 +0200
+++ gtk+-3.8.0/gtk/native/Makefile.in	2013-04-05 02:57:29.304666872 +0300
@@ -405,7 +405,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-@CROSS_COMPILING_TRUE@@USE_EXTERNAL_ICON_CACHE_FALSE@native_update_icon_cache_CFLAGS = $(NATIVE_GDKPIXBUF_CFLAGS)
+@CROSS_COMPILING_TRUE@@USE_EXTERNAL_ICON_CACHE_FALSE@native_update_icon_cache_CFLAGS = $(NATIVE_GDKPIXBUF_CFLAGS) -DNATIVE_BUILD
 @CROSS_COMPILING_TRUE@@USE_EXTERNAL_ICON_CACHE_FALSE@native_update_icon_cache_LDADD = $(NATIVE_GDKPIXBUF_LIBS)
 @CROSS_COMPILING_TRUE@@USE_EXTERNAL_ICON_CACHE_FALSE@native_update_icon_cache_SOURCES = $(srcdir)/../updateiconcache.c
 all: all-am
diff -Nurd gtk+-3.8.0/gtk/updateiconcache.c gtk+-3.8.0/gtk/updateiconcache.c
--- gtk+-3.8.0/gtk/updateiconcache.c	2013-03-17 17:30:02.000000000 +0200
+++ gtk+-3.8.0/gtk/updateiconcache.c	2013-04-05 02:55:31.760669449 +0300
@@ -15,7 +15,9 @@
  * License along with this library. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#ifndef NATIVE_BUILD
 #include "config.h"
+#endif
 
 #include <locale.h>
 #include <stdlib.h>
