diff -Nurd hicolor-icon-theme-0.13/Makefile.in hicolor-icon-theme-0.13/Makefile.in
--- hicolor-icon-theme-0.13/Makefile.in	2013-12-17 11:57:46.000000000 +0200
+++ hicolor-icon-theme-0.13/Makefile.in	2014-10-12 17:36:13.767658080 +0300
@@ -5,6 +5,7 @@
 datadir = @datadir@
 INSTALL = @INSTALL@
 INSTALL_DATA = @INSTALL_DATA@
+srcdir = @srcdir@
 
 distdir = $(PACKAGE)-$(VERSION)
 FILES = README Makefile.in configure.ac configure install-sh index.theme ChangeLog COPYING
@@ -19,7 +20,7 @@
 
 install:
 	mkdir -p $(DESTDIR)$(datadir)/icons/hicolor
-	$(INSTALL_DATA) index.theme $(DESTDIR)$(datadir)/icons/hicolor
+	$(INSTALL_DATA) $(srcdir)/index.theme $(DESTDIR)$(datadir)/icons/hicolor
 	for dir in $(ICON_SIZES); do \
 	  for ctxt in $(ICON_CONTEXTS); do \
 	    mkdir -p $(DESTDIR)$(datadir)/icons/hicolor/$$dir/$$ctxt; \
@@ -42,11 +43,11 @@
 	cvs tag DEFAULT_ICON_THEME_`echo $(VERSION)|sed "s/\./_/"`
 
 # cause rebuilds
-Makefile: Makefile.in config.status
+Makefile: $(srcdir)/Makefile.in config.status
 	./config.status Makefile
-configure: configure.ac
-	autoconf
-config.status: configure
+configure: $(srcdir)/configure.ac
+	cd $(srcdir) && autoconf
+config.status: $(srcdir)/configure
 	./config.status --recheck
 
 .PHONY: all install dist tag
