diff -Nurd ImageMagick-6.9.1-10/magick/locale.c ImageMagick-6.9.1-10/magick/locale.c
--- ImageMagick-6.9.1-10/magick/locale.c	2016-01-04 12:00:02.810988615 +0200
+++ ImageMagick-6.9.1-10/magick/locale.c	2016-01-04 12:02:54.298588992 +0200
@@ -259,7 +259,7 @@
     freelocale(c_locale);
 #elif defined(MAGICKCORE_WINDOWS_SUPPORT)
   if (c_locale != (locale_t) NULL)
-    _free_locale(c_locale);
+    ; //  _free_locale(c_locale); // AcquireLocale() gets never called, so there's nothing to free
 #endif
   c_locale=(locale_t) NULL;
 }
