diff -Nurd bzip2-1.0.5/Makefile bzip2-1.0.5/Makefile
--- bzip2-1.0.5/Makefile	2010-07-23 18:31:26.000000000 +0300
+++ bzip2-1.0.5/Makefile	2010-07-23 18:32:38.000000000 +0300
@@ -14,18 +14,8 @@
 
 SHELL=/bin/sh
 
-# To assist in cross-compiling
-CC=gcc
-AR=ar
-RANLIB=ranlib
-LDFLAGS=
-
 BIGFILES=-D_FILE_OFFSET_BITS=64
-CFLAGS=-Wall -Winline -O2 -g $(BIGFILES)
-
-# Where you want it installed when you do 'make install'
-PREFIX=/usr/local
-
+CFLAGS=$(CPPFLAGS) -Wall -Winline -O2 -g $(BIGFILES)
 
 OBJS= blocksort.o  \
       huffman.o    \
