diff -Nurd gtk+-3.8.1/configure gtk+-3.8.1/configure
--- gtk+-3.8.1/configure	2013-04-15 20:43:50.000000000 +0300
+++ gtk+-3.8.1/configure	2013-04-17 13:40:53.738167756 +0300
@@ -869,6 +869,7 @@
 DISABLE_ON_W32
 GETTEXT_PACKAGE
 LDFLAGS_FOR_BUILD
+LIBS_FOR_BUILD
 CPPFLAGS_FOR_BUILD
 CFLAGS_FOR_BUILD
 BUILD_OBJEXT
@@ -21381,6 +21382,7 @@
    sed -e "/^#/d" -e "/^\$/d" -e "s,.*,	$posrcprefix& \\\\," -e "\$s/\(.*\) \\\\/\1/" \
 	< $srcdir/po/POTFILES.in > po/POTFILES
 
+LIBS_FOR_BUILD=$LIBS
 LIBS="$LIBS $INTLLIBS"
 ac_config_commands="$ac_config_commands po-properties"
 
diff -Nurd gtk+-3.8.1/gtk/native/Makefile.am gtk+-3.8.1/gtk/native/Makefile.am
--- gtk+-3.8.1/gtk/native/Makefile.am	2013-04-17 12:56:28.730226225 +0300
+++ gtk+-3.8.1/gtk/native/Makefile.am	2013-04-17 13:41:12.846167335 +0300
@@ -3,6 +3,7 @@
 CPP = @CPP_FOR_BUILD@
 CPPFLAGS = @CPPFLAGS_FOR_BUILD@
 LDFLAGS = @LDFLAGS_FOR_BUILD@
+LIBS = @LIBS_FOR_BUILD@
 
 if CROSS_COMPILING
 if !USE_EXTERNAL_ICON_CACHE
