diff -Nurd gtk+-2.14.4/modules/other/gail/libgail-util/Makefile.am gtk+-2.14.4/modules/other/gail/libgail-util/Makefile.am
--- gtk+-2.14.4/modules/other/gail/libgail-util/Makefile.am	2008-11-12 10:58:03.000000000 +0200
+++ gtk+-2.14.4/modules/other/gail/libgail-util/Makefile.am	2008-11-12 10:59:08.000000000 +0200
@@ -2,7 +2,7 @@
 
 EXTRA_DIST += gailutil.def
 if OS_WIN32
-export_symbols = -export-symbols gailutil.def
+export_symbols = -export-symbols $(srcdir)/gailutil.def
 no_undefined = -no-undefined
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgailutil.dll.a $(DESTDIR)$(libdir)
@@ -73,7 +73,7 @@
         $(LDFLAGS)
 
 gailutil.lib: libgailutil.la gailutil.def
-	lib -machine:@LIB_EXE_MACHINE_FLAG@ -name:libgailutil-$(GAIL_LT_CURRENT_MINUS_AGE).dll -def:gailutil.def -out:$@
+	lib -machine:@LIB_EXE_MACHINE_FLAG@ -name:libgailutil-$(GAIL_LT_CURRENT_MINUS_AGE).dll -def:$(srcdir)/gailutil.def -out:$@
 
 install-data-local: install-ms-lib install-libtool-import-lib
 
