diff -Nurd -x'*~' freetype-2.3.5/builds/unix/configure.raw freetype-2.3.5/builds/unix/configure.raw
--- freetype-2.3.5/builds/unix/configure.raw	2008-05-05 02:08:28.000000000 +0300
+++ freetype-2.3.5/builds/unix/configure.raw	2008-05-05 02:08:15.000000000 +0300
@@ -494,6 +494,8 @@
 AC_SUBST([FT2_EXTRA_LIBS])
 AC_SUBST([SYSTEM_ZLIB])
 
+AC_LIBTOOL_WIN32_DLL
+
 AC_PROG_LIBTOOL
 # urgh -- these are internal libtool variables...
 AC_SUBST([enable_shared])
