diff -Nurd jpeg-6b/configure jpeg-6b/configure
--- jpeg-6b/configure	1998-03-21 21:08:57.000000000 +0200
+++ jpeg-6b/configure	2008-05-02 20:25:06.000000000 +0300
@@ -1530,6 +1530,7 @@
 if test "x$LTSHARED" != xno  -o  "x$LTSTATIC" != xno; then
   USELIBTOOL="yes"
   LIBTOOL="./libtool"
+  LIBTOOL_CC_TAG="--tag=CC"
   O="lo"
   A="la"
   LN='$(LIBTOOL) --mode=link $(CC)'
@@ -1779,6 +1780,7 @@
 s%@INSTALL_DATA@%$INSTALL_DATA%g
 s%@RANLIB@%$RANLIB%g
 s%@LIBTOOL@%$LIBTOOL%g
+s%@LIBTOOL_CC_TAG@%$LIBTOOL_CC_TAG%g
 s%@O@%$O%g
 s%@A@%$A%g
 s%@LN@%$LN%g
diff -Nurd jpeg-6b/makefile.cfg jpeg-6b/makefile.cfg
--- jpeg-6b/makefile.cfg	1998-03-21 21:08:57.000000000 +0200
+++ jpeg-6b/makefile.cfg	2008-05-02 20:24:10.000000000 +0300
@@ -42,6 +42,8 @@
 O = @O@
 A = @A@
 
+LIBTOOL_CC_TAG = @LIBTOOL_CC_TAG@
+
 # Library version ID; libtool uses this for the shared library version number.
 # Note: we suggest this match the macro of the same name in jpeglib.h.
 JPEG_LIB_VERSION = @JPEG_LIB_VERSION@
@@ -140,7 +142,7 @@
 
 # How to compile with libtool.
 @COM_LT@.c.lo:
-@COM_LT@	$(LIBTOOL) --mode=compile $(CC) $(CFLAGS) -c $(srcdir)/$*.c
+@COM_LT@	$(LIBTOOL) --mode=compile $(LIBTOOL_CC_TAG) $(CC) $(CFLAGS) -c $(srcdir)/$*.c
 
 # How to use ansi2knr, when not using libtool.
 @COM_A2K@.c.o:
@@ -151,7 +153,7 @@
 # How to use ansi2knr AND libtool.
 @COM_A2K@.c.lo:
 @COM_A2K@	./ansi2knr $(srcdir)/$*.c knr/$*.c
-@COM_A2K@	$(LIBTOOL) --mode=compile $(CC) $(CFLAGS) -c knr/$*.c
+@COM_A2K@	$(LIBTOOL) --mode=compile $(LIBTOOL_CC_TAG) $(CC) $(CFLAGS) -c knr/$*.c
 @COM_A2K@	$(RM) knr/$*.c
 
 ansi2knr: ansi2knr.c
@@ -168,7 +170,7 @@
 
 # with libtool:
 libjpeg.la: @A2K_DEPS@ $(LIBOBJECTS)
-	$(LIBTOOL) --mode=link $(CC) -o libjpeg.la $(LIBOBJECTS) \
+	$(LIBTOOL) --mode=link $(LIBTOOL_CC_TAG) $(CC) -o libjpeg.la $(LIBOBJECTS) \
 		-rpath $(libdir) -version-info $(JPEG_LIB_VERSION)
 
 # sample programs:
