diff -Nurd curl-7.46.0/lib/select.h curl-7.46.0/lib/select.h
--- curl-7.46.0/lib/select.h	2015-11-27 17:00:14.000000000 +0200
+++ curl-7.46.0/lib/select.h	2016-01-10 01:31:43.242673744 +0200
@@ -34,6 +34,10 @@
  * Definition of pollfd struct and constants for platforms lacking them.
  */
 
+#if defined(WIN32) && _WIN32_WINNT >= 0x600
+#define HAVE_STRUCT_POLLFD 1
+#endif
+
 #if !defined(HAVE_STRUCT_POLLFD) && \
     !defined(HAVE_SYS_POLL_H) && \
     !defined(HAVE_POLL_H)
