diff -Nurd binutils-2.18/gas/read.c binutils-2.18/gas/read.c
--- binutils-2.18/gas/read.c	2007-08-06 22:59:51.000000000 +0300
+++ binutils-2.18/gas/read.c	2009-05-27 23:21:29.000000000 +0300
@@ -5627,18 +5627,24 @@
       SKIP_WHITESPACE ();
       if (*input_line_pointer != ',')
 	{
+	  int aspret = 0;
+
 	  if (default_prefix)
-	    asprintf (&label, "%s%s", default_prefix, name);
+	    aspret = asprintf (&label, "%s%s", default_prefix, name);
 	  else
 	    {
 	      char leading_char = bfd_get_symbol_leading_char (stdoutput);
 	      /* Missing entry point, use function's name with the leading
 		 char prepended.  */
 	      if (leading_char)
-		asprintf (&label, "%c%s", leading_char, name);
+		aspret = asprintf (&label, "%c%s", leading_char, name);
 	      else
 		label = name;
 	    }
+
+	  if (aspret == -1) {
+	    label = name;
+	  }
 	}
       else
 	{
diff -Nurd binutils-2.18/gas/stabs.c binutils-2.18/gas/stabs.c
--- binutils-2.18/gas/stabs.c	2007-08-28 20:19:36.000000000 +0300
+++ binutils-2.18/gas/stabs.c	2009-05-27 23:24:27.000000000 +0300
@@ -658,6 +658,7 @@
   char *buf;
   char *file;
   unsigned int lineno;
+  int aspret;
 
   if (! void_emitted_p)
     {
@@ -667,11 +668,13 @@
     }
 
   as_where (&file, &lineno);
-  asprintf (&buf, "\"%s:F1\",%d,0,%d,%s",
-	    funcname, N_FUN, lineno + 1, startlabname);
-  input_line_pointer = buf;
-  s_stab ('s');
-  free (buf);
+  aspret = asprintf (&buf, "\"%s:F1\",%d,0,%d,%s",
+		     funcname, N_FUN, lineno + 1, startlabname);
+  if (aspret != -1) {
+    input_line_pointer = buf;
+    s_stab ('s');
+    free (buf);
+  }
 
   input_line_pointer = hold;
   current_function_label = xstrdup (startlabname);
@@ -688,15 +691,18 @@
   char *hold = input_line_pointer;
   char *buf;
   char sym[30];
+  int aspret;
 
   sprintf (sym, "%sendfunc%d", FAKE_LABEL_NAME, label_count);
   ++label_count;
   colon (sym);
 
-  asprintf (&buf, "\"\",%d,0,0,%s-%s", N_FUN, sym, startlabname);
-  input_line_pointer = buf;
-  s_stab ('s');
-  free (buf);
+  aspret = asprintf (&buf, "\"\",%d,0,0,%s-%s", N_FUN, sym, startlabname);
+  if (aspret != -1) {
+    input_line_pointer = buf;
+    s_stab ('s');
+    free (buf);
+  }
 
   input_line_pointer = hold;
   in_dot_func_p = 0;
