diff -Nurd glib-2.14.6/configure.in glib-2.14.6/configure.in
--- glib-2.14.6/configure.in	2008-04-14 17:14:04.000000000 +0300
+++ glib-2.14.6/configure.in	2008-04-14 17:32:31.000000000 +0300
@@ -647,7 +647,9 @@
 	glib_cv_stack_grows=no
         ,
 	glib_cv_stack_grows=yes
-        ,)
+        ,
+        dnl Guess...
+        glib_cv_stack_grows=yes)
 ])
 
 dnl AC_C_INLINE is useless to us since it bails out too early, we need to
