diff -Nurd graphene-1.5.4/configure graphene-1.5.4/configure
--- graphene-1.5.4/configure	2017-01-10 14:49:31.000000000 +0200
+++ graphene-1.5.4/configure	2017-01-17 21:42:24.125841719 +0200
@@ -16972,7 +16972,7 @@
 fi
 
 
-for ac_func in aligned_alloc posix_memalign memalign
+for ac_func in aligned_alloc posix_memalign memalign _aligned_malloc
 do :
   as_ac_var=`$as_echo "ac_cv_func_$ac_func" | $as_tr_sh`
 ac_fn_c_check_func "$LINENO" "$ac_func" "$as_ac_var"
diff -Nurd graphene-1.5.4/configure.ac graphene-1.5.4/configure.ac
--- graphene-1.5.4/configure.ac	2017-01-10 14:49:27.000000000 +0200
+++ graphene-1.5.4/configure.ac	2017-01-17 21:43:31.670379010 +0200
@@ -126,7 +126,7 @@
 AM_CONDITIONAL(OS_LINUX, [test "$platform_linux" = "yes"])
 AM_CONDITIONAL(OS_WIN32, [test "$platform_win32" = "yes"])
 
-AC_CHECK_FUNCS([aligned_alloc posix_memalign memalign])
+AC_CHECK_FUNCS([aligned_alloc posix_memalign memalign _aligned_malloc])
 
 saved_LIBS="$LIBS"
 LIBS=$LIBM
