diff -Nurd harfbuzz-0.9.13/configure.ac harfbuzz-0.9.13/configure.ac
--- harfbuzz-0.9.13/configure.ac	2013-02-26 00:42:35.000000000 +0200
+++ harfbuzz-0.9.13/configure.ac	2013-03-29 20:51:27.030458602 +0200
@@ -171,7 +171,8 @@
 
 dnl ==========================================================================
 
-PKG_CHECK_MODULES(ICU, icu-uc, have_icu=true, have_icu=false)
+dnl PKG_CHECK_MODULES(ICU, icu-uc, have_icu=true, have_icu=false)
+have_icu=false
 if $have_icu; then
 	CXXFLAGS="$CXXFLAGS `$PKG_CONFIG --variable=CXXFLAGS icu-uc`"
 	AC_DEFINE(HAVE_ICU, 1, [Have ICU library])
@@ -180,7 +181,8 @@
 
 dnl ==========================================================================
 
-PKG_CHECK_MODULES(ICU_LE, icu-le icu-uc, have_icu_le=true, have_icu_le=false)
+dnl PKG_CHECK_MODULES(ICU_LE, icu-le icu-uc, have_icu_le=true, have_icu_le=false)
+have_icu_le=false
 if $have_icu_le; then
 	AC_DEFINE(HAVE_ICU_LE, 1, [Have ICU Layout Engine library])
 fi
