diff -Nurd PDCurses-3.4/configure PDCurses-3.4/configure
--- PDCurses-3.4/configure	2008-07-13 04:18:17.000000000 +0300
+++ PDCurses-3.4/configure	2015-06-29 19:28:59.908852127 +0300
@@ -3898,14 +3898,6 @@
 
 fi
 
-
-if test $ac_cv_header_sys_ipc_h = no; then
-	{ { echo "$as_me:$LINENO: error: Cannot find required header file sys/ipc.h; PDCurses cannot be configured" >&5
-echo "$as_me: error: Cannot find required header file sys/ipc.h; PDCurses cannot be configured" >&2;}
-   { (exit 1); exit 1; }; }
-fi
-
-
 if test $ac_cv_header_stdc != yes; then
 	{ { echo "$as_me:$LINENO: error: Need ANSI C headers" >&5
 echo "$as_me: error: Need ANSI C headers" >&2;}
