diff -Nurd libpng-1.6.7/contrib/libtests/pngvalid.c libpng-1.6.7-newbie/contrib/libtests/pngvalid.c
--- libpng-1.6.7/contrib/libtests/pngvalid.c	2013-11-14 21:03:02.000000000 +0200
+++ libpng-1.6.7-newbie/contrib/libtests/pngvalid.c	2013-12-02 23:52:47.443079268 +0200
@@ -43,6 +43,8 @@
 #  include "../../png.h"
 #endif
 
+#define DBL_EPSILON (1e-7)
+
 #ifdef PNG_WRITE_SUPPORTED /* else pngvalid can do nothing */
 
 #if PNG_LIBPNG_VER < 10500
