diff -Nurd SDL2-2.0.3/include/SDL_platform.h SDL2-2.0.3/include/SDL_platform.h
--- SDL2-2.0.3/include/SDL_platform.h	2014-07-23 02:00:05.867196860 +0300
+++ SDL2-2.0.3/include/SDL_platform.h	2014-07-23 02:00:28.075198342 +0300
@@ -117,7 +117,7 @@
 #if defined(WIN32) || defined(_WIN32) || defined(__CYGWIN__)
 /* Try to find out if we're compiling for WinRT or non-WinRT */
 /* If _USING_V110_SDK71_ is defined it means we are using the v110_xp or v120_xp toolset. */
-#if defined(__MINGW32__) || (defined(_MSC_VER) && (_MSC_VER >= 1700) && !_USING_V110_SDK71_)	/* _MSC_VER==1700 for MSVC 2012 */
+#if 0
 #include <winapifamily.h>
 #if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
 #undef __WINDOWS__
