diff -Nurd svgalib-1.9.25/kernel/svgalib_helper/main.c svgalib-1.9.25/kernel/svgalib_helper/main.c
--- svgalib-1.9.25/kernel/svgalib_helper/main.c	2006-01-12 20:17:53.000000000 +0200
+++ svgalib-1.9.25/kernel/svgalib_helper/main.c	2011-05-29 18:02:07.000000000 +0300
@@ -21,7 +21,6 @@
 #include <linux/mm.h>
 #include <linux/thread_info.h>
 #include <linux/smp.h>
-#include <linux/smp_lock.h>
 
 #include <linux/sched.h>
 #include <linux/wait.h>
