diff -Nurd svgalib-1.9.25/kernel/svgalib_helper/Makefile svgalib-1.9.25/kernel/svgalib_helper/Makefile
--- svgalib-1.9.25/kernel/svgalib_helper/Makefile	2006-05-20 15:31:04.000000000 +0300
+++ svgalib-1.9.25/kernel/svgalib_helper/Makefile	2009-08-27 01:57:10.000000000 +0300
@@ -4,10 +4,11 @@
 svgalib_helper-objs := main.o i810.o interrupt.o virtual.o displaystart.o
 obj-m		:= svgalib_helper.o
 
+KERNELRELEASE=<KERNELVER>
 ifneq ($(KERNELRELEASE),)
 
    VER    ?= $(KERNELRELEASE)
-   KDIR	:= /lib/modules/$(VER)/build
+   KDIR	:= <TOPDIR>/lib/modules/$(VER)/build
    PWD	:= $(shell pwd)
    TARGET  := svgalib_helper
 
@@ -18,7 +19,7 @@
 else
 
    VER    ?= $(shell uname -r)
-   KDIR	:= /lib/modules/$(VER)/build
+   KDIR	:= <TOPDIR>/lib/modules/$(VER)/build
    PWD	:= $(shell pwd)
    TARGET  := svgalib_helper
 
@@ -33,12 +34,17 @@
 
 EXTRA_CFLAGS := -DSVGALIB_HELPER_MAJOR=$(SVGALIB_HELPER_MAJOR) $(CLASS_CFLAGS)
 
+ifneq ($(CROSS_COMPILATION),)
+  CROSS_PARAMS=CROSS_COMPILE=<TARGET>- ARCH=<ARCH>
+else
+  CROSS_PARAMS=
+endif
 
 default:
-	$(MAKE) -C $(KDIR) SUBDIRS=$(PWD) CLASS_CFLAGS=$(CLASS_CFLAGS) modules
+	$(MAKE) -C $(KDIR) $(CROSS_PARAMS) SUBDIRS=$(PWD) CLASS_CFLAGS=$(CLASS_CFLAGS) modules
 
 depend:
-	$(MAKE) -C $(KDIR) SUBDIRS=$(PWD) depend
+	$(MAKE) -C $(KDIR) $(CROSS_PARAMS) SUBDIRS=$(PWD) depend
 
 
 clean:
@@ -60,7 +66,7 @@
 install: device modules_install
 
 modules_install: $(MODNAME)
-	mkdir -p /lib/modules/$(VER)/kernel/misc
-	install -m 0644 -c $(TARGET).ko /lib/modules/$(VER)/kernel/misc ||	install -m 0644 -c $(TARGET).o /lib/modules/$(VER)/kernel/misc
-	depmod -a $(VER)
+	mkdir -p <TOPDIR>/lib/modules/$(VER)/kernel/misc
+	install -m 0644 -c $(TARGET).ko <TOPDIR>/lib/modules/$(VER)/kernel/misc ||	install -m 0644 -c $(TARGET).o <TOPDIR>/lib/modules/$(VER)/kernel/misc
+	depmod -a $(VER) -b <TOPDIR>
 
diff -Nurd svgalib-1.9.25/Makefile.cfg svgalib-1.9.25/Makefile.cfg
--- svgalib-1.9.25/Makefile.cfg	2006-07-14 14:38:58.000000000 +0300
+++ svgalib-1.9.25/Makefile.cfg	2009-08-27 01:59:17.000000000 +0300
@@ -19,18 +19,23 @@
 SVGALIB_HELPER_MAJOR = 209
 
 # Linux kernel includes
+S_KERNELRELEASE=<KERNELVER>
 ifeq ($(S_KERNELRELEASE),)
 S_KERNELRELEASE = $(shell uname -r)
 endif
 
-INCLUDEDIR = /lib/modules/$(S_KERNELRELEASE)/build/include
+INCLUDEDIR = <TOPDIR>/lib/modules/$(S_KERNELRELEASE)/build/include
 
 DIREXIST = $(shell if [ -e $(INCLUDEDIR) ] ; then echo OK ; fi)
 ifneq ($(DIREXIST), OK)
 INCLUDEDIR = /usr/src/linux/include
 endif
 
-ARCH := $(shell uname -m | sed -e s/i.86/i386/ -e s/sun4u/sparc64/ -e s/arm.*/arm/ -e s/sa110/arm/)
+ifeq ($(CROSS_COMPILATION),)
+  ARCH := $(shell uname -m | sed -e s/i.86/i386/ -e s/sun4u/sparc64/ -e s/arm.*/arm/ -e s/sa110/arm/)
+else
+  ARCH := <ARCH>
+endif
 
 #----------------------------------------------------------------------
 # Configuration Section
@@ -43,8 +48,8 @@
 
 # Common prefix for installation directories.
 # NOTE: This directory must exist when you start the install.
-TOPDIR=
-prefix = $(TOPDIR)/usr/local
+TOPDIR=<TOPDIR>
+prefix = $(TOPDIR)/usr
 exec_prefix = $(prefix)
 
 # Directory where the shared stubs and static library will be installed.
