diff -Nurd ImageMagick-7.0.1-10/MagickCore/locale.c ImageMagick-7.0.1-10/MagickCore/locale.c
--- ImageMagick-7.0.1-10/MagickCore/locale.c	2016-06-07 16:28:29.000000000 +0300
+++ ImageMagick-7.0.1-10/MagickCore/locale.c	2016-06-14 17:31:26.395778157 +0300
@@ -262,7 +262,7 @@
     freelocale(c_locale);
 #elif defined(MAGICKCORE_WINDOWS_SUPPORT)
   if (c_locale != (locale_t) NULL)
-    _free_locale(c_locale);
+    ; //  _free_locale(c_locale); // AcquireLocale() gets never called, so there's nothing to free
 #endif
   c_locale=(locale_t) NULL;
 }
