diff -Nurd ImageMagick-7.0.1-10/MagickCore/thread_.h ImageMagick-7.0.1-10/MagickCore/thread_.h
--- ImageMagick-7.0.1-10/MagickCore/thread_.h	2016-06-07 16:28:29.000000000 +0300
+++ ImageMagick-7.0.1-10/MagickCore/thread_.h	2016-06-12 11:36:41.082979047 +0300
@@ -23,7 +23,7 @@
 #endif
 
 #if defined(MAGICKCORE_WINDOWS_SUPPORT)
-#include <intsafe.h>
+  // #include <intsafe.h>
 #endif
 
 #if defined(MAGICKCORE_THREAD_SUPPORT)
