diff -Nurd ImageMagick-7.0.1-10/Makefile.am ImageMagick-7.0.1-10/Makefile.am
--- ImageMagick-7.0.1-10/Makefile.am	2016-06-07 16:28:29.000000000 +0300
+++ ImageMagick-7.0.1-10/Makefile.am	2016-06-15 07:06:53.856764516 +0300
@@ -41,9 +41,6 @@
 	$(UTILITIES_CLEANFILES) \
 	$(TESTS_CLEANFILES)
 
-bin_PROGRAMS = \
-	$(UTILITIES_PGMS)
-
 # Binary scripts
 bin_SCRIPTS = \
 	$(MAGICKCORE_BIN_SCRPTS) \
diff -Nurd ImageMagick-7.0.1-10/Makefile.in ImageMagick-7.0.1-10/Makefile.in
--- ImageMagick-7.0.1-10/Makefile.in	2016-06-15 05:59:55.041199452 +0300
+++ ImageMagick-7.0.1-10/Makefile.in	2016-06-15 07:08:14.852585804 +0300
@@ -1801,7 +1801,7 @@
 	$(AM_CFLAGS) $(CFLAGS) $(filters_analyze_la_LDFLAGS) \
 	$(LDFLAGS) -o $@
 @WITH_MODULES_TRUE@am_filters_analyze_la_rpath = -rpath $(filtersdir)
-am__EXEEXT_1 = utilities/magick$(EXEEXT)
+am__EXEEXT_1 =
 am__EXEEXT_2 = tests/validate$(EXEEXT) tests/drawtest$(EXEEXT) \
 	tests/wandtest$(EXEEXT)
 am__EXEEXT_3 = Magick++/demo/analyze$(EXEEXT) \
