diff -Nurd gtk+-3.10.0/gdk/Makefile.am gtk+-3.10.0/gdk/Makefile.am
--- gtk+-3.10.0/gdk/Makefile.am	2013-10-10 03:26:29.899381042 +0300
+++ gtk+-3.10.0/gdk/Makefile.am	2013-10-10 03:28:35.151378294 +0300
@@ -179,8 +179,8 @@
 if USE_WIN32
 libgdk_3_la_SOURCES += gdkkeynames.c
 libgdk_3_la_LIBADD += win32/libgdk-win32.la
-libgdk_3_la_DEPENDENCIES = win32/libgdk-win32.la win32/rc/gdk-win32-res.o gdk.def
-libgdk_3_la_LDFLAGS += -Wl,win32/rc/gdk-win32-res.o -export-symbols $(srcdir)/gdk.def
+libgdk_3_la_DEPENDENCIES = win32/libgdk-win32.la win32/rc/gdk-win32-res.o
+libgdk_3_la_LDFLAGS += -Wl,win32/rc/gdk-win32-res.o
 endif # USE_WIN32
 
 if USE_BROADWAY
@@ -270,22 +270,11 @@
 
 endif # HAVE_INTROSPECTION
 
-if OS_WIN32
-install-def-file: gdk.def
-	mkdir -p $(DESTDIR)$(libdir)
-	$(INSTALL) $(srcdir)/gdk.def $(DESTDIR)$(libdir)/gdk-win32-3.0.def
-uninstall-def-file:
-	-rm $(DESTDIR)$(libdir)/gdk-win32-3.0.def
-else
-install-def-file:
-uninstall-def-file:
-endif
-
 if MS_LIB_AVAILABLE
 noinst_DATA = gdk-win32-$(GTK_API_VERSION).lib
 
-gdk-win32-$(GTK_API_VERSION).lib: libgdk-win32-$(GTK_API_VERSION).la gdk.def
-	lib -machine:@LIB_EXE_MACHINE_FLAG@ -name:libgdk-win32-$(GTK_API_VERSION)-@LT_CURRENT_MINUS_AGE@.dll -def:gdk.def -out:$@
+gdk-win32-$(GTK_API_VERSION).lib: libgdk-win32-$(GTK_API_VERSION).la
+	lib -machine:@LIB_EXE_MACHINE_FLAG@ -name:libgdk-win32-$(GTK_API_VERSION)-@LT_CURRENT_MINUS_AGE@.dll -out:$@
 
 install-ms-lib:
 	mkdir -p $(DESTDIR)$(libdir)
@@ -382,9 +371,9 @@
 
 DISTCLEANFILES = gdkconfig.h stamp-gc-h
 
-install-data-local: install-ms-lib install-def-file
+install-data-local: install-ms-lib
 
-uninstall-local: uninstall-ms-lib uninstall-def-file
+uninstall-local: uninstall-ms-lib
 	rm -f $(DESTDIR)$(configexecincludedir)/gdkconfig.h
 
 # if srcdir!=builddir, clean out maintainer-clean files from builddir
diff -Nurd gtk+-3.10.0/gdk/Makefile.in gtk+-3.10.0/gdk/Makefile.in
--- gtk+-3.10.0/gdk/Makefile.in	2013-10-10 03:26:29.899381042 +0300
+++ gtk+-3.10.0/gdk/Makefile.in	2013-10-10 03:30:15.967376082 +0300
@@ -94,7 +94,7 @@
 @USE_WIN32_FALSE@	$(am__append_1) $(am__append_3) \
 @USE_WIN32_FALSE@	$(am__append_5) $(am__append_7) \
 @USE_WIN32_FALSE@	$(am__append_8)
-@USE_WIN32_TRUE@am__append_6 = -Wl,win32/rc/gdk-win32-res.o -export-symbols $(srcdir)/gdk.def
+@USE_WIN32_TRUE@am__append_6 = -Wl,win32/rc/gdk-win32-res.o
 @USE_BROADWAY_TRUE@am__append_7 = broadway/libgdk-broadway.la
 @USE_WAYLAND_TRUE@am__append_8 = wayland/libgdk-wayland.la
 @HAVE_INTROSPECTION_TRUE@am__append_9 = Gdk-3.0.gir
@@ -736,7 +736,7 @@
 libgdk_3_la_LIBADD = $(GDK_DEP_LIBS) $(am__append_1) $(am__append_3) \
 	$(am__append_5) $(am__append_7) $(am__append_8)
 libgdk_3_la_LDFLAGS = $(LDADD) $(am__append_6)
-@USE_WIN32_TRUE@libgdk_3_la_DEPENDENCIES = win32/libgdk-win32.la win32/rc/gdk-win32-res.o gdk.def
+@USE_WIN32_TRUE@libgdk_3_la_DEPENDENCIES = win32/libgdk-win32.la win32/rc/gdk-win32-res.o
 @HAVE_INTROSPECTION_TRUE@introspection_files = \
 @HAVE_INTROSPECTION_TRUE@	$(filter-out gdkkeysyms-compat.h, $(gdk_public_h_sources))	\
 @HAVE_INTROSPECTION_TRUE@	$(gdk_c_sources)	\
@@ -1624,16 +1624,8 @@
 
 @HAVE_INTROSPECTION_TRUE@@USE_X11_TRUE@GdkX11-3.0.gir: libgdk-3.la Gdk-3.0.gir Makefile
 
-@OS_WIN32_TRUE@install-def-file: gdk.def
-@OS_WIN32_TRUE@	mkdir -p $(DESTDIR)$(libdir)
-@OS_WIN32_TRUE@	$(INSTALL) $(srcdir)/gdk.def $(DESTDIR)$(libdir)/gdk-win32-3.0.def
-@OS_WIN32_TRUE@uninstall-def-file:
-@OS_WIN32_TRUE@	-rm $(DESTDIR)$(libdir)/gdk-win32-3.0.def
-@OS_WIN32_FALSE@install-def-file:
-@OS_WIN32_FALSE@uninstall-def-file:
-
-@MS_LIB_AVAILABLE_TRUE@gdk-win32-$(GTK_API_VERSION).lib: libgdk-win32-$(GTK_API_VERSION).la gdk.def
-@MS_LIB_AVAILABLE_TRUE@	lib -machine:@LIB_EXE_MACHINE_FLAG@ -name:libgdk-win32-$(GTK_API_VERSION)-@LT_CURRENT_MINUS_AGE@.dll -def:gdk.def -out:$@
+@MS_LIB_AVAILABLE_TRUE@gdk-win32-$(GTK_API_VERSION).lib: libgdk-win32-$(GTK_API_VERSION).la
+@MS_LIB_AVAILABLE_TRUE@	lib -machine:@LIB_EXE_MACHINE_FLAG@ -name:libgdk-win32-$(GTK_API_VERSION)-@LT_CURRENT_MINUS_AGE@.dll -out:$@
 
 @MS_LIB_AVAILABLE_TRUE@install-ms-lib:
 @MS_LIB_AVAILABLE_TRUE@	mkdir -p $(DESTDIR)$(libdir)
@@ -1713,9 +1705,9 @@
 	$(CPP) -P - <$(top_srcdir)/build/win32/vs10/gdk.vcxproj.filtersin >$@
 	rm libgdk.vs10.sourcefiles.filters
 
-install-data-local: install-ms-lib install-def-file
+install-data-local: install-ms-lib
 
-uninstall-local: uninstall-ms-lib uninstall-def-file
+uninstall-local: uninstall-ms-lib
 	rm -f $(DESTDIR)$(configexecincludedir)/gdkconfig.h
 
 # if srcdir!=builddir, clean out maintainer-clean files from builddir
