diff -Nurd harfbuzz-0.9.35/src/Makefile.am harfbuzz-0.9.35/src/Makefile.am
--- harfbuzz-0.9.35/src/Makefile.am	2014-08-02 22:00:34.587557683 +0300
+++ harfbuzz-0.9.35/src/Makefile.am	2014-10-09 07:16:12.153564418 +0300
@@ -181,13 +181,8 @@
 harfbuzz_def_dependency = harfbuzz.def
 libharfbuzz_la_LINK = $(CXXLINK) $(libharfbuzz_la_LDFLAGS)
 else
-# Use a C linker for GCC, not C++; Don't link to libstdc++
-if HAVE_GCC
-libharfbuzz_la_LINK = $(LINK) $(libharfbuzz_la_LDFLAGS)
-else
 libharfbuzz_la_LINK = $(CXXLINK) $(libharfbuzz_la_LDFLAGS)
 endif
-endif
 
 libharfbuzz_la_SOURCES = $(HBSOURCES) $(HBHEADERS) $(HBNODISTHEADERS)
 libharfbuzz_la_CPPFLAGS = $(HBCFLAGS)
diff -Nurd harfbuzz-0.9.35/src/Makefile.in harfbuzz-0.9.35/src/Makefile.in
--- harfbuzz-0.9.35/src/Makefile.in	2014-08-13 19:44:10.029190400 +0300
+++ harfbuzz-0.9.35/src/Makefile.in	2014-10-09 07:18:10.529565241 +0300
@@ -916,10 +916,7 @@
 # Put the library together
 @OS_WIN32_TRUE@export_symbols = -export-symbols harfbuzz.def
 @OS_WIN32_TRUE@harfbuzz_def_dependency = harfbuzz.def
-@HAVE_GCC_FALSE@@OS_WIN32_FALSE@libharfbuzz_la_LINK = $(CXXLINK) $(libharfbuzz_la_LDFLAGS)
-# Use a C linker for GCC, not C++; Don't link to libstdc++
-@HAVE_GCC_TRUE@@OS_WIN32_FALSE@libharfbuzz_la_LINK = $(LINK) $(libharfbuzz_la_LDFLAGS)
-@OS_WIN32_TRUE@libharfbuzz_la_LINK = $(CXXLINK) $(libharfbuzz_la_LDFLAGS)
+libharfbuzz_la_LINK = $(CXXLINK) $(libharfbuzz_la_LDFLAGS)
 libharfbuzz_la_SOURCES = $(HBSOURCES) $(HBHEADERS) $(HBNODISTHEADERS)
 libharfbuzz_la_CPPFLAGS = $(HBCFLAGS)
 libharfbuzz_la_LDFLAGS = $(AM_LDFLAGS) -version-info $(HB_LIBTOOL_VERSION_INFO) $(export_symbols) -no-undefined
