diff -Nurd librsvg-2.40.4/rsvg-base.c librsvg-2.40.4/rsvg-base.c
--- librsvg-2.40.4/rsvg-base.c	2014-06-18 21:05:08.000000000 +0300
+++ librsvg-2.40.4/rsvg-base.c	2014-10-12 00:54:50.650020488 +0300
@@ -63,6 +63,8 @@
 #define RSVG_DEFAULT_DPI_X 90.0
 #define RSVG_DEFAULT_DPI_Y 90.0
 
+#define realpath(a,b) _fullpath((b),(a), _MAX_PATH)
+
 G_GNUC_INTERNAL
 double rsvg_internal_dpi_x = RSVG_DEFAULT_DPI_X;
 G_GNUC_INTERNAL
