diff -Nurd gettext-0.19.5.1/gettext-tools/src/Makefile.am gettext-0.19.5.1/gettext-tools/src/Makefile.am
--- gettext-0.19.5.1/gettext-tools/src/Makefile.am	2015-06-23 09:46:19.000000000 +0300
+++ gettext-0.19.5.1/gettext-tools/src/Makefile.am	2015-07-26 06:43:17.307382836 +0300
@@ -315,6 +315,7 @@
 urlget_DEPENDENCIES = $(OTHERPROGDEPENDENCIES)
 
 # Specify installation directory, for --enable-relocatable.
+cldr_plurals_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(bindir)\"
 msgcmp_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(bindir)\"
 msgfmt_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(bindir)\"
 msgmerge_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(bindir)\"
