diff -Nurd ImageMagick-7.0.1-10/Makefile.in ImageMagick-7.0.1-10/Makefile.in
--- ImageMagick-7.0.1-10/Makefile.in	2016-06-07 16:29:00.000000000 +0300
+++ ImageMagick-7.0.1-10/Makefile.in	2016-06-14 17:35:08.203392890 +0300
@@ -764,7 +764,7 @@
 	$(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
 	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(MagickCore_libMagickCore_@MAGICK_MAJOR_VERSION@_@MAGICK_ABI_SUFFIX@_la_LDFLAGS) \
-	$(LDFLAGS) -o $@
+	$(LDFLAGS) -lws2_ws32 -o $@
 MagickWand_libMagickWand_@MAGICK_MAJOR_VERSION@_@MAGICK_ABI_SUFFIX@_la_DEPENDENCIES =  \
 	$(MAGICKCORE_LIBS) $(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
