diff -Nurd gdk-pixbuf-2.24.0/gdk-pixbuf/Makefile.am gdk-pixbuf-2.24.0/gdk-pixbuf/Makefile.am
--- gdk-pixbuf-2.24.0/gdk-pixbuf/Makefile.am	2011-08-16 04:45:14.000000000 +0300
+++ gdk-pixbuf-2.24.0/gdk-pixbuf/Makefile.am	2011-08-31 23:45:03.000000000 +0300
@@ -206,7 +206,7 @@
 gdiplus_ldflag = -Wl,$(libgdiplus)
 
 libgdiplus.dll.a: gdiplus.def
-	$(DLLTOOL) --kill-at --dllname gdiplus.dll --input-def gdiplus.def --output-lib $@
+	$(DLLTOOL) --kill-at --dllname gdiplus.dll --input-def $(srcdir)/gdiplus.def --output-lib $@
 
 if INCLUDE_GDIPLUS
 
