diff -Nurd gettext-0.19.4/gettext-runtime/intl/plural.c gettext-0.19.4/gettext-runtime/intl/plural.c
--- gettext-0.19.4/gettext-runtime/intl/plural.c	2015-01-02 08:51:43.471111089 +0200
+++ gettext-0.19.4/gettext-runtime/intl/plural.c	2015-01-02 08:52:02.127110852 +0200
@@ -310,7 +310,6 @@
 #ifndef YY_
 # if defined YYENABLE_NLS && YYENABLE_NLS
 #  if ENABLE_NLS
-#   include <libintl.h> /* INFRINGES ON USER NAME SPACE */
 #   define YY_(Msgid) dgettext ("bison-runtime", Msgid)
 #  endif
 # endif
