diff -Nurd fontconfig-2.6.0/configure.in fontconfig-2.6.0/configure.in
--- fontconfig-2.6.0/configure.in	2008-06-01 05:24:34.000000000 +0300
+++ fontconfig-2.6.0/configure.in	2009-02-20 18:24:10.000000000 +0200
@@ -35,6 +35,7 @@
 dnl not possible to extract the version number here from fontconfig.h
 AM_INIT_AUTOMAKE(fontconfig, 2.6.0)
 AM_MAINTAINER_MODE
+AC_CANONICAL_HOST
 
 dnl libtool versioning
 
