diff -Nurd gtk+-2.12.9/configure.in gtk+-2.12.9/configure.in
--- gtk+-2.12.9/configure.in	2008-05-07 17:46:48.000000000 +0300
+++ gtk+-2.12.9/configure.in	2008-05-07 18:33:40.000000000 +0300
@@ -139,7 +139,7 @@
 dnl Check for a working C++ compiler, but do not bail out, if none is found.
 dnl We use this for an automated test for C++ header correctness.
 dnl 
-AC_CHECK_PROGS(CXX, [$CCC c++ g++ gcc CC cxx cc++ cl], gcc)
+AC_CHECK_TOOLS(CXX, [$CCC c++ g++ gcc CC cxx cc++ cl], gcc)
 AC_LANG_SAVE
 AC_LANG_CPLUSPLUS
 
