diff -Nurd svgalib-1.9.25/threeDKit/Makefile svgalib-1.9.25/threeDKit/Makefile
--- svgalib-1.9.25/threeDKit/Makefile	2006-07-14 14:30:49.000000000 +0300
+++ svgalib-1.9.25/threeDKit/Makefile	2009-05-29 15:29:41.000000000 +0300
@@ -14,7 +14,7 @@
 # Compiler Section (overrides Makefile.cfg)
 #----------------------------------------------------------------------
 
-INCLUDES += -I$(srcdir)/src
+INCLUDES += -I$(srcdir)/src -I$(srcdir)/gl
 
 #----------------------------------------------------------------------
 # Rules Section
