diff -Nurd jpeg-6b/makefile.cfg jpeg-6b/makefile.cfg
--- jpeg-6b/makefile.cfg	2008-05-02 22:57:51.000000000 +0300
+++ jpeg-6b/makefile.cfg	2008-05-04 16:45:19.000000000 +0300
@@ -170,7 +170,7 @@
 
 # with libtool:
 libjpeg.la: @A2K_DEPS@ $(LIBOBJECTS)
-	$(LIBTOOL) --mode=link $(LIBTOOL_CC_TAG) $(CC) -o libjpeg.la $(LIBOBJECTS) \
+	$(LIBTOOL) --mode=link $(LIBTOOL_CC_TAG) $(CC) -no-undefined -o libjpeg.la $(LIBOBJECTS) \
 		-rpath $(libdir) -version-info $(JPEG_LIB_VERSION)
 
 # sample programs:
