diff -Nurd glib-2.18.0/configure.in glib-2.18.0/configure.in
--- glib-2.18.0/configure.in	2008-09-08 00:46:15.000000000 +0300
+++ glib-2.18.0/configure.in	2008-09-08 00:46:56.000000000 +0300
@@ -572,6 +572,14 @@
 AC_CHECK_SIZEOF(long long)
 AC_CHECK_SIZEOF(__int64)
 
+dnl Some versions of autoconf place newline after value of sizeof variables.
+dnl This causes problems when cross-compiling and target has "\r\n" newline,
+dnl but build system has not. Work around this problem by stripping those
+dnl extra newlines.
+ac_cv_sizeof_long=$(echo $ac_cv_sizeof_long | sed -e 's/\r//g' -e 's/\n//g')
+ac_cv_sizeof_long_long=$(echo $ac_cv_sizeof_long_long | sed -e 's/\r//g' -e 's/\n//g')
+ac_cv_sizeof___int64=$(echo $ac_cv_sizeof___int64 | sed -e 's/\r//g' -e 's/\n//g')
+
 if test x$ac_cv_sizeof_long = x8 || test x$ac_cv_sizeof_long_long = x8 || test x$ac_cv_sizeof___int64 = x8 ; then
   :
 else
