diff -Nurd ImageMagick-6.9.2-10/magick/thread_.h ImageMagick-6.9.2-10/magick/thread_.h
--- ImageMagick-6.9.2-10/magick/thread_.h	2015-12-20 04:23:02.000000000 +0200
+++ ImageMagick-6.9.2-10/magick/thread_.h	2015-12-26 13:49:03.061744673 +0200
@@ -23,7 +23,7 @@
 #endif
 
 #if defined(MAGICKCORE_WINDOWS_SUPPORT)
-#include <intsafe.h>
+  // #include <intsafe.h>
 #endif
 
 #if defined(MAGICKCORE_THREAD_SUPPORT)
