diff -Nurd cairo-1.12.10/config.h.in cairo-1.12.10/config.h.in
--- cairo-1.12.10/config.h.in	2013-01-20 16:30:04.175387963 +0200
+++ cairo-1.12.10/config.h.in	2013-01-20 16:26:14.039388471 +0200
@@ -276,6 +276,8 @@
 /* Define to 1 if you have zlib available */
 #undef HAVE_ZLIB
 
+#undef DBL_EPSILON
+
 /* Define to 1 if the system has the type `__uint128_t'. */
 #undef HAVE___UINT128_T
 
diff -Nurd cairo-1.12.10/configure cairo-1.12.10/configure
--- cairo-1.12.10/configure	2013-01-20 16:30:04.195387963 +0200
+++ cairo-1.12.10/configure	2013-01-20 16:32:32.271387636 +0200
@@ -20314,6 +20314,8 @@
   have_libz="no (requires zlib http://www.gzip.org/zlib/)"
 fi
 
+$as_echo "#define DBL_EPSILON (1e-7)" >> confdefs.h
+
 
 save_LIBS="$LIBS"
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for lzo2a_decompress in -llzo2" >&5
