diff -Nurd freetype-2.5.1/builds/unix/configure.raw freetype-2.5.1/builds/unix/configure.raw
--- freetype-2.5.1/builds/unix/configure.raw	2013-12-15 12:25:51.573211950 +0200
+++ freetype-2.5.1/builds/unix/configure.raw	2013-12-15 12:27:26.017001478 +0200
@@ -287,6 +287,7 @@
                  [do not support png compressed OpenType embedded bitmaps]))
 if test x$with_png != xno; then
   AC_MSG_CHECKING([for libpng])
+  PKG_CHECK_MODULES([LIBPNG], [libpng], [LIBPNG="$LIBPNG_LIBS"])
   if test -z "$LIBPNG_CFLAGS" -a -z "$LIBPNG_LDFLAGS"; then
     if ! which libpng-config >/dev/null; then
       AC_MSG_ERROR([`libpng-config' not found;
@@ -298,9 +299,11 @@
 or pass `--without-png' to the `configure' script.])
     fi
     LIBPNG="`libpng-config --libs`"
+    LIBPNG_LIBS="$LIBPNG"
     LIBPNG_CFLAGS="`libpng-config --cflags`"
     LIBPNG_LDFLAGS="`libpng-config --ldflags`"
   fi
+  LIBPNG_LDFLAGS="$LIBPNG_LIBS"
   HAVE_LIBPNG=yes
   AC_MSG_RESULT([$LIBPNG_LDFLAGS])
 fi
