diff -Nurd pkg-config-0.25/pkg.m4 pkg-config-0.25/pkg.m4
--- pkg-config-0.25/pkg.m4	2009-12-06 22:39:02.000000000 +0200
+++ pkg-config-0.25/pkg.m4	2010-07-17 19:32:46.000000000 +0300
@@ -135,7 +135,7 @@
 Consider adjusting the PKG_CONFIG_PATH environment variable if you
 installed software in a non-standard prefix.
 
-_PKG_TEXT])dnl
+_PKG_TEXT])[]dnl
         ])
 elif test $pkg_failed = untried; then
      	AC_MSG_RESULT([no])
