diff -Nurd tiff-3.8.2/configure.ac tiff-3.8.2/configure.ac
--- tiff-3.8.2/configure.ac	2008-05-13 18:06:51.000000000 +0300
+++ tiff-3.8.2/configure.ac	2008-05-13 18:27:36.000000000 +0300
@@ -30,6 +30,8 @@
 AC_CONFIG_MACRO_DIR(m4)
 AC_LANG(C)
 
+AC_CONFIG_HEADERS([libtiff/tif_config.h libtiff/tiffconf.h])
+
 dnl Compute the canonical target-system type variable
 AC_CANONICAL_TARGET
 
@@ -505,8 +507,6 @@
 
 AC_SUBST(LIBDIR)
 
-AC_CONFIG_HEADERS([libtiff/tif_config.h libtiff/tiffconf.h])
-
 AC_CONFIG_FILES([Makefile \
 		 contrib/Makefile \
 		 contrib/acorn/Makefile \
